{% extends "base.html" %}
{% block head %}
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:dt="uuid:C2F41010-65B3-11d1-A29F-00AA00C14882"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 15">
<meta name=Originator content="Microsoft Word 15">
<link rel=File-List href="RAREFAN_Manual.fld/filelist.xml">
<link rel=Edit-Time-Data href="RAREFAN_Manual.fld/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Frederic Bertels</o:Author>
  <o:LastAuthor>Frederic Bertels</o:LastAuthor>
  <o:Revision>3</o:Revision>
  <o:TotalTime>1</o:TotalTime>
  <o:LastPrinted>2021-09-23T06:19:00Z</o:LastPrinted>
  <o:Created>2021-09-23T06:20:00Z</o:Created>
  <o:LastSaved>2021-09-23T06:20:00Z</o:LastSaved>
  <o:Pages>1</o:Pages>
  <o:Words>5756</o:Words>
  <o:Characters>32814</o:Characters>
  <o:Lines>273</o:Lines>
  <o:Paragraphs>76</o:Paragraphs>
  <o:CharactersWithSpaces>38494</o:CharactersWithSpaces>
  <o:Version>16.00</o:Version>
 </o:DocumentProperties>
 <o:CustomDocumentProperties>
  <o:ZOTERO_PREF_1 dt:dt="string">&lt;data data-version=&quot;3&quot; zotero-version=&quot;5.0.96.3&quot;&gt;&lt;session id=&quot;ni3blpow&quot;/&gt;&lt;style id=&quot;http://www.zotero.org/styles/pnas&quot; hasBibliography=&quot;1&quot; bibliographyStyleHasBeenSet=&quot;1&quot;/&gt;&lt;prefs&gt;&lt;pref name=&quot;fieldType&quot; value=&quot;Field&quot;/&gt;&lt;pref name=&quot;automaticJournalAbbreviati</o:ZOTERO_PREF_1>
  <o:ZOTERO_PREF_2 dt:dt="string">ons&quot; value=&quot;true&quot;/&gt;&lt;/prefs&gt;&lt;/data&gt;</o:ZOTERO_PREF_2>
 </o:CustomDocumentProperties>
 <o:OfficeDocumentSettings>
  <o:AllowPNG/>
 </o:OfficeDocumentSettings>
</xml><![endif]-->
<link rel=themeData href="/static/RAREFAN_Manual.fld/themedata.thmx">
<link rel=colorSchemeMapping href="/static/RAREFAN_Manual.fld/colorschememapping.xml">
<!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:Zoom>127</w:Zoom>
  <w:SpellingState>Clean</w:SpellingState>
  <w:TrackMoves>false</w:TrackMoves>
  <w:TrackFormatting/>
  <w:PunctuationKerning/>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:DoNotPromoteQF/>
  <w:LidThemeOther>EN-GB</w:LidThemeOther>
  <w:LidThemeAsian>X-NONE</w:LidThemeAsian>
  <w:LidThemeComplexScript>X-NONE</w:LidThemeComplexScript>
  <w:Compatibility>
   <w:BreakWrappedTables/>
   <w:SnapToGridInCell/>
   <w:WrapTextWithPunct/>
   <w:UseAsianBreakRules/>
   <w:DontGrowAutofit/>
   <w:SplitPgBreakAndParaMark/>
   <w:EnableOpenTypeKerning/>
   <w:DontFlipMirrorIndents/>
   <w:OverrideTableStyleHps/>
  </w:Compatibility>
  <m:mathPr>
   <m:mathFont m:val="Cambria Math"/>
   <m:brkBin m:val="before"/>
   <m:brkBinSub m:val="&#45;-"/>
   <m:smallFrac m:val="off"/>
   <m:dispDef/>
   <m:lMargin m:val="0"/>
   <m:rMargin m:val="0"/>
   <m:defJc m:val="centerGroup"/>
   <m:wrapIndent m:val="1440"/>
   <m:intLim m:val="subSup"/>
   <m:naryLim m:val="undOvr"/>
  </m:mathPr></w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="false"
  DefSemiHidden="false" DefQFormat="false" DefPriority="99"
  LatentStyleCount="376">
  <w:LsdException Locked="false" Priority="0" QFormat="true" Name="Normal"/>
  <w:LsdException Locked="false" Priority="9" QFormat="true" Name="heading 1"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 2"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 3"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 4"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 5"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 6"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 7"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 8"/>
  <w:LsdException Locked="false" Priority="9" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="heading 9"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 6"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 7"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 8"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index 9"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 1"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 2"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 3"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 4"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 5"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 6"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 7"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 8"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" Name="toc 9"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Normal Indent"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="footnote text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="annotation text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="header"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="footer"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="index heading"/>
  <w:LsdException Locked="false" Priority="35" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="caption"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="table of figures"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="envelope address"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="envelope return"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="footnote reference"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="annotation reference"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="line number"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="page number"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="endnote reference"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="endnote text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="table of authorities"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="macro"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="toa heading"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Bullet"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Number"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Bullet 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Bullet 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Bullet 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Bullet 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Number 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Number 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Number 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Number 5"/>
  <w:LsdException Locked="false" Priority="10" QFormat="true" Name="Title"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Closing"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Signature"/>
  <w:LsdException Locked="false" Priority="1" SemiHidden="true"
   UnhideWhenUsed="true" Name="Default Paragraph Font"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text Indent"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Continue"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Continue 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Continue 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Continue 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="List Continue 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Message Header"/>
  <w:LsdException Locked="false" Priority="11" QFormat="true" Name="Subtitle"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Salutation"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Date"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text First Indent"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text First Indent 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Note Heading"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text Indent 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Body Text Indent 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Block Text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Hyperlink"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="FollowedHyperlink"/>
  <w:LsdException Locked="false" Priority="22" QFormat="true" Name="Strong"/>
  <w:LsdException Locked="false" Priority="20" QFormat="true" Name="Emphasis"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Document Map"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Plain Text"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="E-mail Signature"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Top of Form"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Bottom of Form"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Normal (Web)"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Acronym"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Address"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Cite"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Code"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Definition"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Keyboard"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Preformatted"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Sample"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Typewriter"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="HTML Variable"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Normal Table"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="annotation subject"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="No List"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Outline List 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Outline List 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Outline List 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Simple 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Simple 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Simple 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Classic 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Classic 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Classic 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Classic 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Colorful 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Colorful 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Colorful 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Columns 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Columns 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Columns 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Columns 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Columns 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 6"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 7"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Grid 8"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 4"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 5"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 6"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 7"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table List 8"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table 3D effects 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table 3D effects 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table 3D effects 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Contemporary"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Elegant"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Professional"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Subtle 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Subtle 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Web 1"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Web 2"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Web 3"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Balloon Text"/>
  <w:LsdException Locked="false" Priority="39" Name="Table Grid"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Table Theme"/>
  <w:LsdException Locked="false" SemiHidden="true" Name="Placeholder Text"/>
  <w:LsdException Locked="false" Priority="1" QFormat="true" Name="No Spacing"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 1"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 1"/>
  <w:LsdException Locked="false" SemiHidden="true" Name="Revision"/>
  <w:LsdException Locked="false" Priority="34" QFormat="true"
   Name="List Paragraph"/>
  <w:LsdException Locked="false" Priority="29" QFormat="true" Name="Quote"/>
  <w:LsdException Locked="false" Priority="30" QFormat="true"
   Name="Intense Quote"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 1"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 1"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 1"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 1"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 1"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 2"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 2"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 2"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 2"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 2"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 2"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 3"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 3"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 3"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 3"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 3"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 3"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 4"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 4"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 4"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 4"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 4"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 4"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 5"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 5"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 5"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 5"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 5"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 5"/>
  <w:LsdException Locked="false" Priority="60" Name="Light Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="61" Name="Light List Accent 6"/>
  <w:LsdException Locked="false" Priority="62" Name="Light Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="63" Name="Medium Shading 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="64" Name="Medium Shading 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="65" Name="Medium List 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="66" Name="Medium List 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="67" Name="Medium Grid 1 Accent 6"/>
  <w:LsdException Locked="false" Priority="68" Name="Medium Grid 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="69" Name="Medium Grid 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="70" Name="Dark List Accent 6"/>
  <w:LsdException Locked="false" Priority="71" Name="Colorful Shading Accent 6"/>
  <w:LsdException Locked="false" Priority="72" Name="Colorful List Accent 6"/>
  <w:LsdException Locked="false" Priority="73" Name="Colorful Grid Accent 6"/>
  <w:LsdException Locked="false" Priority="19" QFormat="true"
   Name="Subtle Emphasis"/>
  <w:LsdException Locked="false" Priority="21" QFormat="true"
   Name="Intense Emphasis"/>
  <w:LsdException Locked="false" Priority="31" QFormat="true"
   Name="Subtle Reference"/>
  <w:LsdException Locked="false" Priority="32" QFormat="true"
   Name="Intense Reference"/>
  <w:LsdException Locked="false" Priority="33" QFormat="true" Name="Book Title"/>
  <w:LsdException Locked="false" Priority="37" SemiHidden="true"
   UnhideWhenUsed="true" Name="Bibliography"/>
  <w:LsdException Locked="false" Priority="39" SemiHidden="true"
   UnhideWhenUsed="true" QFormat="true" Name="TOC Heading"/>
  <w:LsdException Locked="false" Priority="41" Name="Plain Table 1"/>
  <w:LsdException Locked="false" Priority="42" Name="Plain Table 2"/>
  <w:LsdException Locked="false" Priority="43" Name="Plain Table 3"/>
  <w:LsdException Locked="false" Priority="44" Name="Plain Table 4"/>
  <w:LsdException Locked="false" Priority="45" Name="Plain Table 5"/>
  <w:LsdException Locked="false" Priority="40" Name="Grid Table Light"/>
  <w:LsdException Locked="false" Priority="46" Name="Grid Table 1 Light"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark"/>
  <w:LsdException Locked="false" Priority="51" Name="Grid Table 6 Colorful"/>
  <w:LsdException Locked="false" Priority="52" Name="Grid Table 7 Colorful"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 1"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 1"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 1"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 1"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 1"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 2"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 2"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 2"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 2"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 2"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 3"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 3"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 3"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 3"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 3"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 4"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 4"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 4"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 4"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 4"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 5"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 5"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 5"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 5"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 5"/>
  <w:LsdException Locked="false" Priority="46"
   Name="Grid Table 1 Light Accent 6"/>
  <w:LsdException Locked="false" Priority="47" Name="Grid Table 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="48" Name="Grid Table 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="49" Name="Grid Table 4 Accent 6"/>
  <w:LsdException Locked="false" Priority="50" Name="Grid Table 5 Dark Accent 6"/>
  <w:LsdException Locked="false" Priority="51"
   Name="Grid Table 6 Colorful Accent 6"/>
  <w:LsdException Locked="false" Priority="52"
   Name="Grid Table 7 Colorful Accent 6"/>
  <w:LsdException Locked="false" Priority="46" Name="List Table 1 Light"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark"/>
  <w:LsdException Locked="false" Priority="51" Name="List Table 6 Colorful"/>
  <w:LsdException Locked="false" Priority="52" Name="List Table 7 Colorful"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 1"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 1"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 1"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 1"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 1"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 1"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 1"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 2"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 2"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 2"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 2"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 2"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 2"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 2"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 3"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 3"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 3"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 3"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 3"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 3"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 3"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 4"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 4"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 4"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 4"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 4"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 4"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 4"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 5"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 5"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 5"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 5"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 5"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 5"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 5"/>
  <w:LsdException Locked="false" Priority="46"
   Name="List Table 1 Light Accent 6"/>
  <w:LsdException Locked="false" Priority="47" Name="List Table 2 Accent 6"/>
  <w:LsdException Locked="false" Priority="48" Name="List Table 3 Accent 6"/>
  <w:LsdException Locked="false" Priority="49" Name="List Table 4 Accent 6"/>
  <w:LsdException Locked="false" Priority="50" Name="List Table 5 Dark Accent 6"/>
  <w:LsdException Locked="false" Priority="51"
   Name="List Table 6 Colorful Accent 6"/>
  <w:LsdException Locked="false" Priority="52"
   Name="List Table 7 Colorful Accent 6"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Mention"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Smart Hyperlink"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Hashtag"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Unresolved Mention"/>
  <w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"
   Name="Smart Link"/>
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Courier;
	panose-1:0 0 0 0 0 0 0 0 0 0;
	mso-font-charset:0;
	mso-generic-font-family:auto;
	mso-font-pitch:variable;
	mso-font-signature:3 0 0 0 3 0;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:roman;
	mso-font-pitch:variable;
	mso-font-signature:-536869121 1107305727 33554432 0 415 0;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:swiss;
	mso-font-pitch:variable;
	mso-font-signature:-536859905 -1073697537 9 0 511 0;}
@font-face
	{font-family:"Calibri Light";
	panose-1:2 15 3 2 2 2 4 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:swiss;
	mso-font-pitch:variable;
	mso-font-signature:-469750017 -1073732485 9 0 511 0;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin:0cm;
	line-height:150%;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;
	mso-fareast-font-family:"Times New Roman";}
h1
	{mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-link:"Heading 1 Char";
	mso-style-next:Normal;
	margin-top:12.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	line-height:150%;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:1;
	font-size:20.0pt;
	mso-bidi-font-size:16.0pt;
	font-family:"Calibri Light",sans-serif;
	mso-ascii-font-family:"Calibri Light";
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:"Calibri Light";
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:#2F5496;
	mso-themecolor:accent1;
	mso-themeshade:191;
	mso-font-kerning:0pt;
	font-weight:normal;}
h2
	{mso-style-priority:9;
	mso-style-qformat:yes;
	mso-style-link:"Heading 2 Char";
	mso-style-next:Normal;
	margin-top:2.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	line-height:150%;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:2;
	font-size:14.0pt;
	mso-bidi-font-size:13.0pt;
	font-family:"Calibri Light",sans-serif;
	mso-ascii-font-family:"Calibri Light";
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:"Calibri Light";
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:#2F5496;
	mso-themecolor:accent1;
	mso-themeshade:191;
	mso-fareast-language:EN-US;
	font-weight:normal;}
h3
	{mso-style-noshow:yes;
	mso-style-priority:9;
	mso-style-qformat:yes;
	mso-style-link:"Heading 3 Char";
	mso-style-next:Normal;
	margin-top:2.0pt;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:0cm;
	line-height:150%;
	mso-pagination:widow-orphan lines-together;
	page-break-after:avoid;
	mso-outline-level:3;
	font-size:12.0pt;
	font-family:"Calibri Light",sans-serif;
	mso-ascii-font-family:"Calibri Light";
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:"Calibri Light";
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:#1F3763;
	mso-themecolor:accent1;
	mso-themeshade:127;
	font-weight:normal;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	text-decoration:underline;
	text-underline:single;}
a:visited, span.MsoHyperlinkFollowed
	{mso-style-noshow:yes;
	mso-style-priority:99;
	color:#954F72;
	mso-themecolor:followedhyperlink;
	text-decoration:underline;
	text-underline:single;}
p
	{mso-style-priority:99;
	mso-margin-top-alt:auto;
	margin-right:0cm;
	mso-margin-bottom-alt:auto;
	margin-left:0cm;
	line-height:150%;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;
	mso-fareast-font-family:"Times New Roman";}
p.MsoListParagraph, li.MsoListParagraph, div.MsoListParagraph
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	mso-add-space:auto;
	line-height:150%;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;
	mso-fareast-font-family:"Times New Roman";}
p.MsoListParagraphCxSpFirst, li.MsoListParagraphCxSpFirst, div.MsoListParagraphCxSpFirst
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-type:export-only;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	mso-add-space:auto;
	line-height:150%;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;
	mso-fareast-font-family:"Times New Roman";}
p.MsoListParagraphCxSpMiddle, li.MsoListParagraphCxSpMiddle, div.MsoListParagraphCxSpMiddle
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-type:export-only;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	mso-add-space:auto;
	line-height:150%;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;
	mso-fareast-font-family:"Times New Roman";}
p.MsoListParagraphCxSpLast, li.MsoListParagraphCxSpLast, div.MsoListParagraphCxSpLast
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-type:export-only;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	mso-add-space:auto;
	line-height:150%;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;
	mso-fareast-font-family:"Times New Roman";}
p.MsoBibliography, li.MsoBibliography, div.MsoBibliography
	{mso-style-priority:37;
	mso-style-next:Normal;
	margin-top:0cm;
	margin-right:0cm;
	margin-bottom:12.0pt;
	margin-left:19.2pt;
	text-indent:-19.2pt;
	line-height:150%;
	mso-pagination:widow-orphan;
	tab-stops:19.0pt;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;
	mso-fareast-font-family:"Times New Roman";}
span.Heading1Char
	{mso-style-name:"Heading 1 Char";
	mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Heading 1";
	mso-ansi-font-size:20.0pt;
	mso-bidi-font-size:16.0pt;
	font-family:"Calibri Light",sans-serif;
	mso-ascii-font-family:"Calibri Light";
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:"Calibri Light";
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:#2F5496;
	mso-themecolor:accent1;
	mso-themeshade:191;}
span.Heading3Char
	{mso-style-name:"Heading 3 Char";
	mso-style-noshow:yes;
	mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Heading 3";
	font-family:"Calibri Light",sans-serif;
	mso-ascii-font-family:"Calibri Light";
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:"Calibri Light";
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:#1F3763;
	mso-themecolor:accent1;
	mso-themeshade:127;}
span.Heading2Char
	{mso-style-name:"Heading 2 Char";
	mso-style-priority:9;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Heading 2";
	mso-ansi-font-size:14.0pt;
	mso-bidi-font-size:13.0pt;
	font-family:"Calibri Light",sans-serif;
	mso-ascii-font-family:"Calibri Light";
	mso-ascii-theme-font:major-latin;
	mso-fareast-font-family:"Times New Roman";
	mso-fareast-theme-font:major-fareast;
	mso-hansi-font-family:"Calibri Light";
	mso-hansi-theme-font:major-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:major-bidi;
	color:#2F5496;
	mso-themecolor:accent1;
	mso-themeshade:191;}
span.SpellE
	{mso-style-name:"";
	mso-spl-e:yes;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Calibri;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-fareast-language:EN-US;}
@page WordSection1
	{size:612.0pt 792.0pt;
	margin:72.0pt 72.0pt 72.0pt 72.0pt;
	mso-header-margin:35.4pt;
	mso-footer-margin:35.4pt;
	mso-paper-source:0;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 @list l0
	{mso-list-id:533419405;
	mso-list-type:hybrid;
	mso-list-template-ids:749876970 882914398 134807577 134807579 134807567 134807577 134807579 134807567 134807577 134807579;}
@list l0:level1
	{mso-level-text:"\(%1\)";
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;
	color:red;
	mso-ansi-font-style:normal;
	mso-bidi-font-style:normal;}
@list l0:level2
	{mso-level-number-format:alpha-lower;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l0:level3
	{mso-level-number-format:roman-lower;
	mso-level-tab-stop:none;
	mso-level-number-position:right;
	text-indent:-9.0pt;}
@list l0:level4
	{mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l0:level5
	{mso-level-number-format:alpha-lower;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l0:level6
	{mso-level-number-format:roman-lower;
	mso-level-tab-stop:none;
	mso-level-number-position:right;
	text-indent:-9.0pt;}
@list l0:level7
	{mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l0:level8
	{mso-level-number-format:alpha-lower;
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-18.0pt;}
@list l0:level9
	{mso-level-number-format:roman-lower;
	mso-level-tab-stop:none;
	mso-level-number-position:right;
	text-indent:-9.0pt;}
ol
	{margin-bottom:0cm;}
ul
	{margin-bottom:0cm;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
 table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-parent:"";
	mso-padding-alt:0cm 5.4pt 0cm 5.4pt;
	mso-para-margin:0cm;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Calibri",sans-serif;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;
	mso-fareast-language:EN-US;}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:shapedefaults v:ext="edit" spidmax="1038"/>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <o:shapelayout v:ext="edit">
  <o:idmap v:ext="edit" data="1"/>
 </o:shapelayout></xml><![endif]-->
</head>
{% endblock %}

{% block body %}
<body lang=EN-GB link=blue vlink="#954F72" style='tab-interval:36.0pt;
word-wrap:break-word'>

<div class=WordSection1>

<h1><span lang=EN-US style='mso-ansi-language:EN-US'>RAREFAN (<span
class=SpellE>RAyt</span>/<span class=SpellE>REpin</span> Finder and <span
class=SpellE>ANalyzer</span>) Manual<o:p></o:p></span></h1>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-US style='mso-ansi-language:EN-US'>Using the online tool<o:p></o:p></span></h2>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US;mso-no-proof:
yes'><!--[if gte vml 1]><v:shapetype id="_x0000_t75" coordsize="21600,21600"
 o:spt="75" o:preferrelative="t" path="m@4@5l@4@11@9@11@9@5xe" filled="f"
 stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="Picture_x0020_1" o:spid="_x0000_i1031" type="#_x0000_t75"
 style='width:312pt;height:227pt;visibility:visible;mso-wrap-style:square'
 o:bordertopcolor="yellow pure" o:borderleftcolor="yellow pure"
 o:borderbottomcolor="yellow pure" o:borderrightcolor="yellow pure">
 <v:imagedata src="/static/RAREFAN_Manual.fld/image001.png" o:title=""/>
 <w:bordertop type="single" width="6"/>
 <w:borderleft type="single" width="6"/>
 <w:borderbottom type="single" width="6"/>
 <w:borderright type="single" width="6"/>
</v:shape><![endif]--><![if !vml]><img width=314 height=229
src="/static/RAREFAN_Manual.fld/image026.png" v:shapes="Picture_x0020_1"><![endif]></span><span
lang=EN-US style='mso-ansi-language:EN-US'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'>Here you
can upload the genomes that are to be analyzed for their REPIN content. The
genomes should be in <span class=SpellE>fasta</span> format and only have the
specified file extensions. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'>You can
also upload RAYT query sequences. These should be <span class=SpellE>fasta</span>
files containing <span class=SpellE>aminoacid</span> sequences and should end
in “.fa” or “.<span class=SpellE>faa</span>”. If these are not provided then
you can select a RAYT gene from <i>P. fluorescens </i>SBW25 (Group 3 RAYT </span><!--[if supportFields]><span
lang=EN-US style='mso-ansi-language:EN-US'><span style='mso-element:field-begin'></span><span
style='mso-spacerun:yes'> </span>ADDIN ZOTERO_ITEM CSL_CITATION
{&quot;citationID&quot;:&quot;9iQyIYzh&quot;,&quot;properties&quot;:{&quot;formattedCitation&quot;:&quot;(1)&quot;,&quot;plainCitation&quot;:&quot;(1)&quot;,&quot;noteIndex&quot;:0},&quot;citationItems&quot;:[{&quot;id&quot;:539,&quot;uris&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/Z6EAGJX4&quot;],&quot;uri&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/Z6EAGJX4&quot;],&quot;itemData&quot;:{&quot;id&quot;:539,&quot;type&quot;:&quot;article-journal&quot;,&quot;abstract&quot;:&quot;Selfish
genetic elements, such as insertion sequences and transposons are found in most
genomes. Transposons are usually identifiable by their high copy number within
genomes. In contrast, REP-associated tyrosine transposases (RAYTs), a recently
described class of bacterial transposase, are typically present at just one
copy per genome. This suggests that RAYTs no longer copy themselves and thus
they no longer function as a typical transposase. Motivated by this possibility
we interrogated thousands of fully sequenced bacterial genomes in order to
determine patterns of RAYT diversity, their distribution across chromosomes and
accessory elements, and rate of duplication. RAYTs encompass exceptional
diversity and are divisible into at least five distinct groups. They possess features
more similar to housekeeping genes than insertion sequences, are predominantly
vertically transmitted and have persisted through evolutionary time to the
point where they are now found in 24% of all species for which at least one
fully sequenced genome is available. Overall, the genomic distribution of RAYTs
suggests that they have been coopted by host genomes to perform a function that
benefits the host cell.&quot;,&quot;container-title&quot;:&quot;Genome Biology
and Evolution&quot;,&quot;DOI&quot;:&quot;10.1093/gbe/evx146&quot;,&quot;issue&quot;:&quot;8&quot;,&quot;journalAbbreviation&quot;:&quot;Genome
Biol.
Evol.&quot;,&quot;page&quot;:&quot;2110-2121&quot;,&quot;title&quot;:&quot;Identification
and Characterization of Domesticated Bacterial
Transposases.&quot;,&quot;volume&quot;:&quot;9&quot;,&quot;author&quot;:[{&quot;family&quot;:&quot;Bertels&quot;,&quot;given&quot;:&quot;Frederic&quot;},{&quot;family&quot;:&quot;Gallie&quot;,&quot;given&quot;:&quot;Jenna&quot;},{&quot;family&quot;:&quot;Rainey&quot;,&quot;given&quot;:&quot;Paul
B&quot;}],&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2017&quot;,8,1]]}}}],&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}
<span style='mso-element:field-separator'></span></span><![endif]--><span
lang=EN-US style='mso-ansi-language:EN-US'><span style='mso-no-proof:yes'>(1)</span></span><!--[if supportFields]><span
lang=EN-US style='mso-ansi-language:EN-US'><span style='mso-element:field-end'></span></span><![endif]--><span
lang=EN-US style='mso-ansi-language:EN-US'>) or one from <i>E. coli</i> (Group
2 RAYT </span><!--[if supportFields]><span lang=EN-US style='mso-ansi-language:
EN-US'><span style='mso-element:field-begin'></span><span
style='mso-spacerun:yes'> </span>ADDIN ZOTERO_ITEM CSL_CITATION
{&quot;citationID&quot;:&quot;Lvxhn9DX&quot;,&quot;properties&quot;:{&quot;formattedCitation&quot;:&quot;(1)&quot;,&quot;plainCitation&quot;:&quot;(1)&quot;,&quot;noteIndex&quot;:0},&quot;citationItems&quot;:[{&quot;id&quot;:539,&quot;uris&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/Z6EAGJX4&quot;],&quot;uri&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/Z6EAGJX4&quot;],&quot;itemData&quot;:{&quot;id&quot;:539,&quot;type&quot;:&quot;article-journal&quot;,&quot;abstract&quot;:&quot;Selfish
genetic elements, such as insertion sequences and transposons are found in most
genomes. Transposons are usually identifiable by their high copy number within
genomes. In contrast, REP-associated tyrosine transposases (RAYTs), a recently
described class of bacterial transposase, are typically present at just one
copy per genome. This suggests that RAYTs no longer copy themselves and thus
they no longer function as a typical transposase. Motivated by this possibility
we interrogated thousands of fully sequenced bacterial genomes in order to
determine patterns of RAYT diversity, their distribution across chromosomes and
accessory elements, and rate of duplication. RAYTs encompass exceptional
diversity and are divisible into at least five distinct groups. They possess
features more similar to housekeeping genes than insertion sequences, are
predominantly vertically transmitted and have persisted through evolutionary
time to the point where they are now found in 24% of all species for which at
least one fully sequenced genome is available. Overall, the genomic
distribution of RAYTs suggests that they have been coopted by host genomes to
perform a function that benefits the host
cell.&quot;,&quot;container-title&quot;:&quot;Genome Biology and
Evolution&quot;,&quot;DOI&quot;:&quot;10.1093/gbe/evx146&quot;,&quot;issue&quot;:&quot;8&quot;,&quot;journalAbbreviation&quot;:&quot;Genome
Biol.
Evol.&quot;,&quot;page&quot;:&quot;2110-2121&quot;,&quot;title&quot;:&quot;Identification
and Characterization of Domesticated Bacterial
Transposases.&quot;,&quot;volume&quot;:&quot;9&quot;,&quot;author&quot;:[{&quot;family&quot;:&quot;Bertels&quot;,&quot;given&quot;:&quot;Frederic&quot;},{&quot;family&quot;:&quot;Gallie&quot;,&quot;given&quot;:&quot;Jenna&quot;},{&quot;family&quot;:&quot;Rainey&quot;,&quot;given&quot;:&quot;Paul
B&quot;}],&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2017&quot;,8,1]]}}}],&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}
<span style='mso-element:field-separator'></span></span><![endif]--><span
lang=EN-US style='mso-ansi-language:EN-US'><span style='mso-no-proof:yes'>(1)</span></span><!--[if supportFields]><span
lang=EN-US style='mso-ansi-language:EN-US'><span style='mso-element:field-end'></span></span><![endif]--><span
lang=EN-US style='mso-ansi-language:EN-US'>) provided by the server on the next
screen.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'>Finally, it
is possible to upload a phylogenetic tree in <span class=SpellE>Newick</span>
format for your <span class=SpellE>fasta</span> sequences. For the webserver to
be able to use the provided tree the leaves of the tree have to match the
sequence file names. If the tree is not provided then a tree will be
reconstructed using <span class=SpellE>andi</span> </span><!--[if supportFields]><span
lang=EN-US style='mso-ansi-language:EN-US'><span style='mso-element:field-begin'></span><span
style='mso-spacerun:yes'> </span>ADDIN ZOTERO_ITEM CSL_CITATION
{&quot;citationID&quot;:&quot;iLJcETAs&quot;,&quot;properties&quot;:{&quot;formattedCitation&quot;:&quot;(2)&quot;,&quot;plainCitation&quot;:&quot;(2)&quot;,&quot;noteIndex&quot;:0},&quot;citationItems&quot;:[{&quot;id&quot;:156,&quot;uris&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/GZ74BZ5Z&quot;],&quot;uri&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/GZ74BZ5Z&quot;],&quot;itemData&quot;:{&quot;id&quot;:156,&quot;type&quot;:&quot;article-journal&quot;,&quot;abstract&quot;:&quot;MOTIVATION:A
standard approach to classifying sets of genomes is to calculate their pairwise
distances. This is difficult for large samples. We have therefore developed an
algorithm for rapidly computing the evolutionary distances between closely
related genomes.\n\nRESULTS:Our distance measure is based on ungapped local
alignments that we anchor through pairs of maximal unique matches of a minimum
length. These exact matches can be looked up efficiently using enhanced suffix
arrays and our implementation requires approximately only 1 s and 45 MB
RAM/Mbase analysed. The pairing of matches distinguishes non-homologous from
homologous regions leading to accurate distance estimation. We show this by
analysing simulated data and genome samples ranging from 29 Escherichia
coli/Shigella genomes to 3085 genomes of Streptococcus
pneumoniae.\n\nAVAILABILITY AND IMPLEMENTATION:We have implemented the
computation of anchor distances in the multithreaded UNIX command-line program
andi for ANchor DIstances. C sources and documentation are posted at http://github.com/evolbioinf/andi/\n\nCONTACT:haubold@evolbio.mpg.de\n\nSUPPLEMENTARY
INFORMATION:Supplementary data are available at Bioinformatics
online.&quot;,&quot;container-title&quot;:&quot;Bioinformatics&quot;,&quot;DOI&quot;:&quot;10.1093/bioinformatics/btu815&quot;,&quot;issue&quot;:&quot;8&quot;,&quot;journalAbbreviation&quot;:&quot;Bioinformatics&quot;,&quot;page&quot;:&quot;1169-1175&quot;,&quot;title&quot;:&quot;andi:
fast and accurate estimation of evolutionary distances between closely related
genomes.&quot;,&quot;volume&quot;:&quot;31&quot;,&quot;author&quot;:[{&quot;family&quot;:&quot;Haubold&quot;,&quot;given&quot;:&quot;Bernhard&quot;},{&quot;family&quot;:&quot;Klötzl&quot;,&quot;given&quot;:&quot;Fabian&quot;},{&quot;family&quot;:&quot;Pfaffelhuber&quot;,&quot;given&quot;:&quot;Peter&quot;}],&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2015&quot;,4,15]]}}}],&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}
<span style='mso-element:field-separator'></span></span><![endif]--><span
lang=EN-US style='mso-ansi-language:EN-US'><span style='mso-no-proof:yes'>(2)</span></span><!--[if supportFields]><span
lang=EN-US style='mso-ansi-language:EN-US'><span style='mso-element:field-end'></span></span><![endif]--><span
lang=EN-US style='mso-ansi-language:EN-US'>. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoListParagraph style='margin-left:269.35pt;mso-add-space:auto;
text-indent:-18.0pt;mso-list:l0 level1 lfo1;tab-stops:290.6pt'><![if !supportLists]><span
lang=EN-US style='color:red;mso-ansi-language:EN-US'><span style='mso-list:
Ignore'>(1)<span style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><![endif]><!--[if gte vml 1]><v:shapetype
 id="_x0000_t202" coordsize="21600,21600" o:spt="202" path="m,l,21600r21600,l21600,xe">
 <v:stroke joinstyle="miter"/>
 <v:path gradientshapeok="t" o:connecttype="rect"/>
</v:shapetype><v:shape id="Text_x0020_Box_x0020_17" o:spid="_x0000_s1037"
 type="#_x0000_t202" style='position:absolute;left:0;text-align:left;
 margin-left:99.7pt;margin-top:281.85pt;width:28.4pt;height:24.55pt;
 text-indent:0;z-index:251673600;visibility:visible;mso-wrap-style:none;
 mso-height-percent:0;mso-wrap-distance-left:9pt;mso-wrap-distance-top:0;
 mso-wrap-distance-right:9pt;mso-wrap-distance-bottom:0;
 mso-position-horizontal:absolute;mso-position-horizontal-relative:text;
 mso-position-vertical:absolute;mso-position-vertical-relative:text;
 mso-height-percent:0;mso-height-relative:margin;v-text-anchor:top' o:gfxdata="UEsDBBQABgAIAAAAIQC2gziS/gAAAOEBAAATAAAAW0NvbnRlbnRfVHlwZXNdLnhtbJSRQU7DMBBF
90jcwfIWJU67QAgl6YK0S0CoHGBkTxKLZGx5TGhvj5O2G0SRWNoz/78nu9wcxkFMGNg6quQqL6RA
0s5Y6ir5vt9lD1JwBDIwOMJKHpHlpr69KfdHjyxSmriSfYz+USnWPY7AufNIadK6MEJMx9ApD/oD
OlTrorhX2lFEilmcO2RdNtjC5xDF9pCuTyYBB5bi6bQ4syoJ3g9WQ0ymaiLzg5KdCXlKLjvcW893
SUOqXwnz5DrgnHtJTxOsQfEKIT7DmDSUCaxw7Rqn8787ZsmRM9e2VmPeBN4uqYvTtW7jvijg9N/y
JsXecLq0q+WD6m8AAAD//wMAUEsDBBQABgAIAAAAIQA4/SH/1gAAAJQBAAALAAAAX3JlbHMvLnJl
bHOkkMFqwzAMhu+DvYPRfXGawxijTi+j0GvpHsDYimMaW0Yy2fr2M4PBMnrbUb/Q94l/f/hMi1qR
JVI2sOt6UJgd+ZiDgffL8ekFlFSbvV0oo4EbChzGx4f9GRdb25HMsYhqlCwG5lrLq9biZkxWOiqY
22YiTra2kYMu1l1tQD30/bPm3wwYN0x18gb45AdQl1tp5j/sFB2T0FQ7R0nTNEV3j6o9feQzro1i
OWA14Fm+Q8a1a8+Bvu/d/dMb2JY5uiPbhG/ktn4cqGU/er3pcvwCAAD//wMAUEsDBBQABgAIAAAA
IQBSp4/3PQIAAHgEAAAOAAAAZHJzL2Uyb0RvYy54bWysVE1v2zAMvQ/YfxB0X+2kaZsZcYosRYYB
RVugHXpWZDkxIIuCpMTufv2e5KRNu52GXWSKpB75+OHZdd9qtlfON2RKPjrLOVNGUtWYTcl/Pq2+
TDnzQZhKaDKq5C/K8+v550+zzhZqTFvSlXIMIMYXnS35NgRbZJmXW9UKf0ZWGRhrcq0IuLpNVjnR
Ab3V2TjPL7OOXGUdSeU9tDeDkc8Tfl0rGe7r2qvAdMmRW0inS+c6ntl8JoqNE3bbyEMa4h+yaEVj
EPQV6kYEwXau+QOqbaQjT3U4k9RmVNeNVIkD2IzyD2wet8KqxAXF8fa1TP7/wcq7/YNjTYXeXXFm
RIsePak+sG/UM6hQn876Am6PFo6hhx6+R72HMtLua9fGLwgx2FHpl9fqRjQJ5dfRZJLDImE6H42m
02lEyd4eW+fDd0Uti0LJHZqXair2tz4MrkeXGMuTbqpVo3W6xIFRS+3YXqDVOqQUAf7OSxvWlfzy
/CJPwIbi8wFZG+QSqQ6UohT6dX/gv6bqBfQdDQPkrVw1SPJW+PAgHCYGvLAF4R5HrQlB6CBxtiX3
62/66I9GwspZhwksucGKcKZ/GDQ4VQsDmy6Ti6sxIrhTy/rUYnbtksB7hG2zMonRP+ijWDtqn7Eq
ixgTJmEkIpc8HMVlGLYCqybVYpGcMKJWhFvzaGWEjnWODXjqn4Wzhy4FtPeOjpMqig/NGnzjS0OL
XaC6SZ2M5R1qeqg6xjvNwmEV4/6c3pPX2w9j/hsAAP//AwBQSwMEFAAGAAgAAAAhAOaOnFnkAAAA
EAEAAA8AAABkcnMvZG93bnJldi54bWxMT8tOwzAQvCPxD9YicaPOowQnjVOhoko90AMBxNWNTRIR
r0Pstunfs5zgstLszs6jXM92YCcz+d6hhHgRATPYON1jK+HtdXsngPmgUKvBoZFwMR7W1fVVqQrt
zvhiTnVoGYmgL5SELoSx4Nw3nbHKL9xokG6fbrIqEJxarid1JnE78CSKMm5Vj+TQqdFsOtN81Ucr
Yb/Ja7FLLtNHnu62tfiO3bN4l/L2Zn5a0XhcAQtmDn8f8NuB8kNFwQ7uiNqzgXCeL4kq4T5LH4AR
I12mtDlIyOJEAK9K/r9I9QMAAP//AwBQSwECLQAUAAYACAAAACEAtoM4kv4AAADhAQAAEwAAAAAA
AAAAAAAAAAAAAAAAW0NvbnRlbnRfVHlwZXNdLnhtbFBLAQItABQABgAIAAAAIQA4/SH/1gAAAJQB
AAALAAAAAAAAAAAAAAAAAC8BAABfcmVscy8ucmVsc1BLAQItABQABgAIAAAAIQBSp4/3PQIAAHgE
AAAOAAAAAAAAAAAAAAAAAC4CAABkcnMvZTJvRG9jLnhtbFBLAQItABQABgAIAAAAIQDmjpxZ5AAA
ABABAAAPAAAAAAAAAAAAAAAAAJcEAABkcnMvZG93bnJldi54bWxQSwUGAAAAAAQABADzAAAAqAUA
AAAA
" fillcolor="white [3201]" stroked="f" strokeweight=".5pt">
 <v:textbox>
  <![if !mso]>
  <table cellpadding=0 cellspacing=0 width="100%">
   <tr>
    <td><![endif]>
    <div>
    <p class=MsoNormal><span lang=EN-US style='color:red;mso-ansi-language:
    EN-US'>(8)<o:p></o:p></span></p>
    </div>
    <![if !mso]></td>
   </tr>
  </table>
  <![endif]></v:textbox>
</v:shape><![endif]--><![if !vml]><span style='mso-ignore:vglayout;position:
absolute;z-index:251673600;left:0px;margin-left:90px;margin-top:93px;
width:32px;height:28px'><img width=32 height=28
src="/static/RAREFAN_Manual.fld/image027.png" alt="Text Box: (8)" v:shapes="Text_x0020_Box_x0020_17"></span><![endif]><!--[if gte vml 1]><v:shape
 id="Text_x0020_Box_x0020_15" o:spid="_x0000_s1036" type="#_x0000_t202"
 style='position:absolute;left:0;text-align:left;margin-left:77.95pt;
 margin-top:252.25pt;width:28.4pt;height:24.55pt;text-indent:0;z-index:251671552;
 visibility:visible;mso-wrap-style:none;mso-height-percent:0;
 mso-wrap-distance-left:9pt;mso-wrap-distance-top:0;mso-wrap-distance-right:9pt;
 mso-wrap-distance-bottom:0;mso-position-horizontal:absolute;
 mso-position-horizontal-relative:text;mso-position-vertical:absolute;
 mso-position-vertical-relative:text;mso-height-percent:0;
 mso-height-relative:margin;v-text-anchor:top' o:gfxdata="UEsDBBQABgAIAAAAIQC2gziS/gAAAOEBAAATAAAAW0NvbnRlbnRfVHlwZXNdLnhtbJSRQU7DMBBF
90jcwfIWJU67QAgl6YK0S0CoHGBkTxKLZGx5TGhvj5O2G0SRWNoz/78nu9wcxkFMGNg6quQqL6RA
0s5Y6ir5vt9lD1JwBDIwOMJKHpHlpr69KfdHjyxSmriSfYz+USnWPY7AufNIadK6MEJMx9ApD/oD
OlTrorhX2lFEilmcO2RdNtjC5xDF9pCuTyYBB5bi6bQ4syoJ3g9WQ0ymaiLzg5KdCXlKLjvcW893
SUOqXwnz5DrgnHtJTxOsQfEKIT7DmDSUCaxw7Rqn8787ZsmRM9e2VmPeBN4uqYvTtW7jvijg9N/y
JsXecLq0q+WD6m8AAAD//wMAUEsDBBQABgAIAAAAIQA4/SH/1gAAAJQBAAALAAAAX3JlbHMvLnJl
bHOkkMFqwzAMhu+DvYPRfXGawxijTi+j0GvpHsDYimMaW0Yy2fr2M4PBMnrbUb/Q94l/f/hMi1qR
JVI2sOt6UJgd+ZiDgffL8ekFlFSbvV0oo4EbChzGx4f9GRdb25HMsYhqlCwG5lrLq9biZkxWOiqY
22YiTra2kYMu1l1tQD30/bPm3wwYN0x18gb45AdQl1tp5j/sFB2T0FQ7R0nTNEV3j6o9feQzro1i
OWA14Fm+Q8a1a8+Bvu/d/dMb2JY5uiPbhG/ktn4cqGU/er3pcvwCAAD//wMAUEsDBBQABgAIAAAA
IQCWmD7BPwIAAH8EAAAOAAAAZHJzL2Uyb0RvYy54bWysVMFu2zAMvQ/YPwi6r07StMuMOEXWIsOA
oi2QDD0rshwbkEVBUmN3X78nOUmzbqdhF5kiqUfykfT8pm812yvnGzIFH1+MOFNGUtmYXcF/bFaf
Zpz5IEwpNBlV8Ffl+c3i44d5Z3M1oZp0qRwDiPF5Zwteh2DzLPOyVq3wF2SVgbEi14qAq9tlpRMd
0FudTUaj66wjV1pHUnkP7d1g5IuEX1VKhseq8iowXXDkFtLp0rmNZ7aYi3znhK0beUhD/EMWrWgM
gp6g7kQQ7MU1f0C1jXTkqQoXktqMqqqRKtWAasajd9Wsa2FVqgXkeHuiyf8/WPmwf3KsKdG7K86M
aNGjjeoD+0o9gwr8dNbncFtbOIYeevge9R7KWHZfuTZ+URCDHUy/ntiNaBLKL+PpdASLhOlyPJ7N
ZhEle3tsnQ/fFLUsCgV3aF7iVOzvfRhcjy4xlifdlKtG63SJA6NutWN7gVbrkFIE+G9e2rCu4NeX
V6MEbCg+H5C1QS6x1KGkKIV+2w/UHMvdUvkKFhwNc+StXDXI9V748CQcBgflYRnCI45KE2LRQeKs
Jvfzb/roj37CylmHQSy4waZwpr8b9DmRhrlNl+nV5wkiuHPL9txiXtpbQvljLJ2VSYz+QR/FylH7
jI1ZxpgwCSMRueDhKN6GYTmwcVItl8kJk2pFuDdrKyN0pDv2YdM/C2cPzQro8gMdB1bk73o2+MaX
hpYvgaomNTSyPHB6IB9TnkbisJFxjc7vyevtv7H4BQAA//8DAFBLAwQUAAYACAAAACEAkGNT+uQA
AAAQAQAADwAAAGRycy9kb3ducmV2LnhtbExPQU7DMBC8I/EHa5G4UacpqeI0ToWKKvUABwJVr268
JBGxHWy3TX/PcoLLSjM7OztTriczsDP60DsrYT5LgKFtnO5tK+HjffuQAwtRWa0GZ1HCFQOsq9ub
UhXaXewbnuvYMjKxoVASuhjHgvPQdGhUmLkRLe0+nTcqEvQt115dyNwMPE2SJTeqt/ShUyNuOmy+
6pOR8LoRdb5Lr/4gFrttnX/P3Uu+l/L+bnpe0XhaAYs4xb8L+O1A+aGiYEd3sjqwgXCWCZJKyJLH
DBgpUiGIORKTLZbAq5L/L1L9AAAA//8DAFBLAQItABQABgAIAAAAIQC2gziS/gAAAOEBAAATAAAA
AAAAAAAAAAAAAAAAAABbQ29udGVudF9UeXBlc10ueG1sUEsBAi0AFAAGAAgAAAAhADj9If/WAAAA
lAEAAAsAAAAAAAAAAAAAAAAALwEAAF9yZWxzLy5yZWxzUEsBAi0AFAAGAAgAAAAhAJaYPsE/AgAA
fwQAAA4AAAAAAAAAAAAAAAAALgIAAGRycy9lMm9Eb2MueG1sUEsBAi0AFAAGAAgAAAAhAJBjU/rk
AAAAEAEAAA8AAAAAAAAAAAAAAAAAmQQAAGRycy9kb3ducmV2LnhtbFBLBQYAAAAABAAEAPMAAACq
BQAAAAA=
" fillcolor="white [3201]" stroked="f" strokeweight=".5pt">
 <v:textbox>
  <![if !mso]>
  <table cellpadding=0 cellspacing=0 width="100%">
   <tr>
    <td><![endif]>
    <div>
    <p class=MsoNormal><span lang=EN-US style='color:red;mso-ansi-language:
    EN-US'>(7)<o:p></o:p></span></p>
    </div>
    <![if !mso]></td>
   </tr>
  </table>
  <![endif]></v:textbox>
</v:shape><![endif]--><![if !vml]><span style='mso-ignore:vglayout;position:
absolute;z-index:251671552;left:0px;margin-left:68px;margin-top:63px;
width:32px;height:29px'><img width=32 height=29
src="/static/RAREFAN_Manual.fld/image028.png" alt="Text Box: (7)" v:shapes="Text_x0020_Box_x0020_15"></span><![endif]><!--[if gte vml 1]><v:shape
 id="Text_x0020_Box_x0020_14" o:spid="_x0000_s1035" type="#_x0000_t202"
 style='position:absolute;left:0;text-align:left;margin-left:80.35pt;
 margin-top:221pt;width:28.4pt;height:24.55pt;text-indent:0;z-index:251669504;
 visibility:visible;mso-wrap-style:none;mso-height-percent:0;
 mso-wrap-distance-left:9pt;mso-wrap-distance-top:0;mso-wrap-distance-right:9pt;
 mso-wrap-distance-bottom:0;mso-position-horizontal:absolute;
 mso-position-horizontal-relative:text;mso-position-vertical:absolute;
 mso-position-vertical-relative:text;mso-height-percent:0;
 mso-height-relative:margin;v-text-anchor:top' o:gfxdata="UEsDBBQABgAIAAAAIQC2gziS/gAAAOEBAAATAAAAW0NvbnRlbnRfVHlwZXNdLnhtbJSRQU7DMBBF
90jcwfIWJU67QAgl6YK0S0CoHGBkTxKLZGx5TGhvj5O2G0SRWNoz/78nu9wcxkFMGNg6quQqL6RA
0s5Y6ir5vt9lD1JwBDIwOMJKHpHlpr69KfdHjyxSmriSfYz+USnWPY7AufNIadK6MEJMx9ApD/oD
OlTrorhX2lFEilmcO2RdNtjC5xDF9pCuTyYBB5bi6bQ4syoJ3g9WQ0ymaiLzg5KdCXlKLjvcW893
SUOqXwnz5DrgnHtJTxOsQfEKIT7DmDSUCaxw7Rqn8787ZsmRM9e2VmPeBN4uqYvTtW7jvijg9N/y
JsXecLq0q+WD6m8AAAD//wMAUEsDBBQABgAIAAAAIQA4/SH/1gAAAJQBAAALAAAAX3JlbHMvLnJl
bHOkkMFqwzAMhu+DvYPRfXGawxijTi+j0GvpHsDYimMaW0Yy2fr2M4PBMnrbUb/Q94l/f/hMi1qR
JVI2sOt6UJgd+ZiDgffL8ekFlFSbvV0oo4EbChzGx4f9GRdb25HMsYhqlCwG5lrLq9biZkxWOiqY
22YiTra2kYMu1l1tQD30/bPm3wwYN0x18gb45AdQl1tp5j/sFB2T0FQ7R0nTNEV3j6o9feQzro1i
OWA14Fm+Q8a1a8+Bvu/d/dMb2JY5uiPbhG/ktn4cqGU/er3pcvwCAAD//wMAUEsDBBQABgAIAAAA
IQBuQBANQAIAAH8EAAAOAAAAZHJzL2Uyb0RvYy54bWysVFFv2jAQfp+0/2D5fQQo7RgiVIyKaVLV
VoKpz8ZxIJLjs2xDwn79PjtAWbenaS/O+e783d13d5net7VmB+V8RSbng16fM2UkFZXZ5vzHevlp
zJkPwhRCk1E5PyrP72cfP0wbO1FD2pEulGMAMX7S2JzvQrCTLPNyp2rhe2SVgbEkV4uAq9tmhRMN
0GudDfv9u6whV1hHUnkP7UNn5LOEX5ZKhuey9CownXPkFtLp0rmJZzabisnWCbur5CkN8Q9Z1KIy
CHqBehBBsL2r/oCqK+nIUxl6kuqMyrKSKtWAagb9d9WsdsKqVAvI8fZCk/9/sPLp8OJYVaB3I86M
qNGjtWoD+0otgwr8NNZP4LaycAwt9PA96z2Usey2dHX8oiAGO5g+XtiNaBLKL4PRqA+LhOlmMBiP
xxEle3tsnQ/fFNUsCjl3aF7iVBwefehczy4xliddFctK63SJA6MW2rGDQKt1SCkC/DcvbViT87ub
234CNhSfd8jaIJdYaldSlEK7aRM1w3O5GyqOYMFRN0feymWFXB+FDy/CYXBQHpYhPOMoNSEWnSTO
duR+/k0f/dFPWDlrMIg5N9gUzvR3gz4n0jC36TK6/TxEBHdt2VxbzL5eEMofYOmsTGL0D/oslo7q
V2zMPMaESRiJyDkPZ3ERuuXAxkk1nycnTKoV4dGsrIzQke7Yh3X7Kpw9NSugy090Hlgxedezzje+
NDTfByqr1NDIcsfpiXxMeRqJ00bGNbq+J6+3/8bsFwAAAP//AwBQSwMEFAAGAAgAAAAhAFob4Tvk
AAAAEAEAAA8AAABkcnMvZG93bnJldi54bWxMT0FOwzAQvCPxB2uRuFE7aVSSNE6Fiir1AAcCiKsb
myQiXgfbbdPfs5zgstLMzs7OVJvZjuxkfBgcSkgWApjB1ukBOwlvr7u7HFiICrUaHRoJFxNgU19f
VarU7owv5tTEjpEJhlJJ6GOcSs5D2xurwsJNBmn36bxVkaDvuPbqTOZ25KkQK27VgPShV5PZ9qb9
ao5WwvO2aPJ9evEfxXK/a/LvxD3l71Le3syPaxoPa2DRzPHvAn47UH6oKdjBHVEHNhJeiXuSSsiy
lJqRYikyYg7EFEkCvK74/yL1DwAAAP//AwBQSwECLQAUAAYACAAAACEAtoM4kv4AAADhAQAAEwAA
AAAAAAAAAAAAAAAAAAAAW0NvbnRlbnRfVHlwZXNdLnhtbFBLAQItABQABgAIAAAAIQA4/SH/1gAA
AJQBAAALAAAAAAAAAAAAAAAAAC8BAABfcmVscy8ucmVsc1BLAQItABQABgAIAAAAIQBuQBANQAIA
AH8EAAAOAAAAAAAAAAAAAAAAAC4CAABkcnMvZTJvRG9jLnhtbFBLAQItABQABgAIAAAAIQBaG+E7
5AAAABABAAAPAAAAAAAAAAAAAAAAAJoEAABkcnMvZG93bnJldi54bWxQSwUGAAAAAAQABADzAAAA
qwUAAAAA
" fillcolor="white [3201]" stroked="f" strokeweight=".5pt">
 <v:textbox>
  <![if !mso]>
  <table cellpadding=0 cellspacing=0 width="100%">
   <tr>
    <td><![endif]>
    <div>
    <p class=MsoNormal><span lang=EN-US style='color:red;mso-ansi-language:
    EN-US'>(6)<o:p></o:p></span></p>
    </div>
    <![if !mso]></td>
   </tr>
  </table>
  <![endif]></v:textbox>
</v:shape><![endif]--><![if !vml]><span style='mso-ignore:vglayout;position:
absolute;z-index:251669504;left:0px;margin-left:70px;margin-top:32px;
width:33px;height:29px'><img width=33 height=29
src="/static/RAREFAN_Manual.fld/image029.png" alt="Text Box: (6)" v:shapes="Text_x0020_Box_x0020_14"></span><![endif]><!--[if gte vml 1]><v:shape
 id="Text_x0020_Box_x0020_13" o:spid="_x0000_s1034" type="#_x0000_t202"
 style='position:absolute;left:0;text-align:left;margin-left:80.35pt;
 margin-top:184.65pt;width:28.4pt;height:24.55pt;text-indent:0;z-index:251667456;
 visibility:visible;mso-wrap-style:none;mso-height-percent:0;
 mso-wrap-distance-left:9pt;mso-wrap-distance-top:0;mso-wrap-distance-right:9pt;
 mso-wrap-distance-bottom:0;mso-position-horizontal:absolute;
 mso-position-horizontal-relative:text;mso-position-vertical:absolute;
 mso-position-vertical-relative:text;mso-height-percent:0;
 mso-height-relative:margin;v-text-anchor:top' o:gfxdata="UEsDBBQABgAIAAAAIQC2gziS/gAAAOEBAAATAAAAW0NvbnRlbnRfVHlwZXNdLnhtbJSRQU7DMBBF
90jcwfIWJU67QAgl6YK0S0CoHGBkTxKLZGx5TGhvj5O2G0SRWNoz/78nu9wcxkFMGNg6quQqL6RA
0s5Y6ir5vt9lD1JwBDIwOMJKHpHlpr69KfdHjyxSmriSfYz+USnWPY7AufNIadK6MEJMx9ApD/oD
OlTrorhX2lFEilmcO2RdNtjC5xDF9pCuTyYBB5bi6bQ4syoJ3g9WQ0ymaiLzg5KdCXlKLjvcW893
SUOqXwnz5DrgnHtJTxOsQfEKIT7DmDSUCaxw7Rqn8787ZsmRM9e2VmPeBN4uqYvTtW7jvijg9N/y
JsXecLq0q+WD6m8AAAD//wMAUEsDBBQABgAIAAAAIQA4/SH/1gAAAJQBAAALAAAAX3JlbHMvLnJl
bHOkkMFqwzAMhu+DvYPRfXGawxijTi+j0GvpHsDYimMaW0Yy2fr2M4PBMnrbUb/Q94l/f/hMi1qR
JVI2sOt6UJgd+ZiDgffL8ekFlFSbvV0oo4EbChzGx4f9GRdb25HMsYhqlCwG5lrLq9biZkxWOiqY
22YiTra2kYMu1l1tQD30/bPm3wwYN0x18gb45AdQl1tp5j/sFB2T0FQ7R0nTNEV3j6o9feQzro1i
OWA14Fm+Q8a1a8+Bvu/d/dMb2JY5uiPbhG/ktn4cqGU/er3pcvwCAAD//wMAUEsDBBQABgAIAAAA
IQAjfFA4PwIAAH8EAAAOAAAAZHJzL2Uyb0RvYy54bWysVMFu2zAMvQ/YPwi6r46TtMuCOEXWosOA
oC2QDj0rshwbkEVBUmNnX78nOUm7bqdhF5kiqUfykfTium812yvnGzIFzy9GnCkjqWzMruA/nu4+
zTjzQZhSaDKq4Afl+fXy44dFZ+dqTDXpUjkGEOPnnS14HYKdZ5mXtWqFvyCrDIwVuVYEXN0uK53o
gN7qbDwaXWUdudI6ksp7aG8HI18m/KpSMjxUlVeB6YIjt5BOl85tPLPlQsx3Tti6kcc0xD9k0YrG
IOgZ6lYEwV5c8wdU20hHnqpwIanNqKoaqVINqCYfvatmUwurUi0gx9szTf7/wcr7/aNjTYneTTgz
okWPnlQf2FfqGVTgp7N+DreNhWPooYfvSe+hjGX3lWvjFwUx2MH04cxuRJNQfsmn0xEsEqZJns9m
s4iSvT62zodviloWhYI7NC9xKvZrHwbXk0uM5Uk35V2jdbrEgVE32rG9QKt1SCkC/DcvbVhX8KvJ
5SgBG4rPB2RtkEssdSgpSqHf9omaMw1bKg9gwdEwR97Kuwa5roUPj8JhcFAeliE84Kg0IRYdJc5q
cj//po/+6CesnHUYxIIbbApn+rtBnxNpmNt0mV5+HiOCe2vZvrWYl/aGUH6OpbMyidE/6JNYOWqf
sTGrGBMmYSQiFzycxJswLAc2TqrVKjlhUq0Ia7OxMkJHumMfnvpn4eyxWQFdvqfTwIr5u54NvvGl
odVLoKpJDY0sD5weyceUp5E4bmRco7f35PX631j+AgAA//8DAFBLAwQUAAYACAAAACEA9lPiH+QA
AAAQAQAADwAAAGRycy9kb3ducmV2LnhtbExPPU/DMBDdkfgP1iGxUTtNFJw0ToVaVeoAAwHE6sYm
iYjt1Hbb9N9zTLCc9O7evY9qPZuRnLUPg7MCkgUDom3r1GA7Ae9vuwcOJERplRyd1QKuOsC6vr2p
ZKncxb7qcxM7giI2lFJAH+NUUhraXhsZFm7SFm9fzhsZEfqOKi8vKG5GumQsp0YOFh16OelNr9vv
5mQEvGyKhu+XV/9ZpPtdw4+Je+YfQtzfzdsVjqcVkKjn+PcBvx0wP9QY7OBOVgUyIs7ZI1IFpHmR
AkFGyjLcHARkCc+A1hX9X6T+AQAA//8DAFBLAQItABQABgAIAAAAIQC2gziS/gAAAOEBAAATAAAA
AAAAAAAAAAAAAAAAAABbQ29udGVudF9UeXBlc10ueG1sUEsBAi0AFAAGAAgAAAAhADj9If/WAAAA
lAEAAAsAAAAAAAAAAAAAAAAALwEAAF9yZWxzLy5yZWxzUEsBAi0AFAAGAAgAAAAhACN8UDg/AgAA
fwQAAA4AAAAAAAAAAAAAAAAALgIAAGRycy9lMm9Eb2MueG1sUEsBAi0AFAAGAAgAAAAhAPZT4h/k
AAAAEAEAAA8AAAAAAAAAAAAAAAAAmQQAAGRycy9kb3ducmV2LnhtbFBLBQYAAAAABAAEAPMAAACq
BQAAAAA=
" fillcolor="white [3201]" stroked="f" strokeweight=".5pt">
 <v:textbox>
  <![if !mso]>
  <table cellpadding=0 cellspacing=0 width="100%">
   <tr>
    <td><![endif]>
    <div>
    <p class=MsoNormal><span lang=EN-US style='color:red;mso-ansi-language:
    EN-US'>(5)<o:p></o:p></span></p>
    </div>
    <![if !mso]></td>
   </tr>
  </table>
  <![endif]></v:textbox>
</v:shape><![endif]--><![if !vml]><span style='mso-ignore:vglayout;position:
relative;z-index:251667456'><span style='left:0px;position:absolute;left:70px;
top:-4px;width:33px;height:28px'><img width=33 height=28
src="/static/RAREFAN_Manual.fld/image030.png" alt="Text Box: (5)" v:shapes="Text_x0020_Box_x0020_13"></span></span><![endif]><!--[if gte vml 1]><v:shape
 id="Text_x0020_Box_x0020_12" o:spid="_x0000_s1033" type="#_x0000_t202"
 style='position:absolute;left:0;text-align:left;margin-left:80.35pt;
 margin-top:147.35pt;width:28.4pt;height:24.55pt;text-indent:0;z-index:251665408;
 visibility:visible;mso-wrap-style:none;mso-height-percent:0;
 mso-wrap-distance-left:9pt;mso-wrap-distance-top:0;mso-wrap-distance-right:9pt;
 mso-wrap-distance-bottom:0;mso-position-horizontal:absolute;
 mso-position-horizontal-relative:text;mso-position-vertical:absolute;
 mso-position-vertical-relative:text;mso-height-percent:0;
 mso-height-relative:margin;v-text-anchor:top' o:gfxdata="UEsDBBQABgAIAAAAIQC2gziS/gAAAOEBAAATAAAAW0NvbnRlbnRfVHlwZXNdLnhtbJSRQU7DMBBF
90jcwfIWJU67QAgl6YK0S0CoHGBkTxKLZGx5TGhvj5O2G0SRWNoz/78nu9wcxkFMGNg6quQqL6RA
0s5Y6ir5vt9lD1JwBDIwOMJKHpHlpr69KfdHjyxSmriSfYz+USnWPY7AufNIadK6MEJMx9ApD/oD
OlTrorhX2lFEilmcO2RdNtjC5xDF9pCuTyYBB5bi6bQ4syoJ3g9WQ0ymaiLzg5KdCXlKLjvcW893
SUOqXwnz5DrgnHtJTxOsQfEKIT7DmDSUCaxw7Rqn8787ZsmRM9e2VmPeBN4uqYvTtW7jvijg9N/y
JsXecLq0q+WD6m8AAAD//wMAUEsDBBQABgAIAAAAIQA4/SH/1gAAAJQBAAALAAAAX3JlbHMvLnJl
bHOkkMFqwzAMhu+DvYPRfXGawxijTi+j0GvpHsDYimMaW0Yy2fr2M4PBMnrbUb/Q94l/f/hMi1qR
JVI2sOt6UJgd+ZiDgffL8ekFlFSbvV0oo4EbChzGx4f9GRdb25HMsYhqlCwG5lrLq9biZkxWOiqY
22YiTra2kYMu1l1tQD30/bPm3wwYN0x18gb45AdQl1tp5j/sFB2T0FQ7R0nTNEV3j6o9feQzro1i
OWA14Fm+Q8a1a8+Bvu/d/dMb2JY5uiPbhG/ktn4cqGU/er3pcvwCAAD//wMAUEsDBBQABgAIAAAA
IQDoOnPqQAIAAH8EAAAOAAAAZHJzL2Uyb0RvYy54bWysVFFv2jAQfp+0/2D5fQQo7RgiVIyKaVLV
VoKpz8ZxIJLjs2xDwn79PjtAWbenaS/O+e783d13d5net7VmB+V8RSbng16fM2UkFZXZ5vzHevlp
zJkPwhRCk1E5PyrP72cfP0wbO1FD2pEulGMAMX7S2JzvQrCTLPNyp2rhe2SVgbEkV4uAq9tmhRMN
0GudDfv9u6whV1hHUnkP7UNn5LOEX5ZKhuey9CownXPkFtLp0rmJZzabisnWCbur5CkN8Q9Z1KIy
CHqBehBBsL2r/oCqK+nIUxl6kuqMyrKSKtWAagb9d9WsdsKqVAvI8fZCk/9/sPLp8OJYVaB3Q86M
qNGjtWoD+0otgwr8NNZP4LaycAwt9PA96z2Usey2dHX8oiAGO5g+XtiNaBLKL4PRqA+LhOlmMBiP
xxEle3tsnQ/fFNUsCjl3aF7iVBwefehczy4xliddFctK63SJA6MW2rGDQKt1SCkC/DcvbViT87ub
234CNhSfd8jaIJdYaldSlEK7aRM1o3O5GyqOYMFRN0feymWFXB+FDy/CYXBQHpYhPOMoNSEWnSTO
duR+/k0f/dFPWDlrMIg5N9gUzvR3gz4n0jC36TK6/TxEBHdt2VxbzL5eEMofYOmsTGL0D/oslo7q
V2zMPMaESRiJyDkPZ3ERuuXAxkk1nycnTKoV4dGsrIzQke7Yh3X7Kpw9NSugy090Hlgxedezzje+
NDTfByqr1NDIcsfpiXxMeRqJ00bGNbq+J6+3/8bsFwAAAP//AwBQSwMEFAAGAAgAAAAhAN48+K7j
AAAAEAEAAA8AAABkcnMvZG93bnJldi54bWxMT0FOwzAQvCPxB2uRuFG7SVWSNE6Fiir1AIcGEFc3
NklEvA6226a/Z3uCy2pGOzs7U64nO7CT8aF3KGE+E8AMNk732Ep4f9s+ZMBCVKjV4NBIuJgA6+r2
plSFdmfcm1MdW0YmGAoloYtxLDgPTWesCjM3GqTdl/NWRaK+5dqrM5nbgSdCLLlVPdKHTo1m05nm
uz5aCa+bvM52ycV/5uluW2c/c/eSfUh5fzc9r2g8rYBFM8W/C7h2oPxQUbCDO6IObCC+FI8klZDk
CwKkSMUVHAgs0gx4VfL/RapfAAAA//8DAFBLAQItABQABgAIAAAAIQC2gziS/gAAAOEBAAATAAAA
AAAAAAAAAAAAAAAAAABbQ29udGVudF9UeXBlc10ueG1sUEsBAi0AFAAGAAgAAAAhADj9If/WAAAA
lAEAAAsAAAAAAAAAAAAAAAAALwEAAF9yZWxzLy5yZWxzUEsBAi0AFAAGAAgAAAAhAOg6c+pAAgAA
fwQAAA4AAAAAAAAAAAAAAAAALgIAAGRycy9lMm9Eb2MueG1sUEsBAi0AFAAGAAgAAAAhAN48+K7j
AAAAEAEAAA8AAAAAAAAAAAAAAAAAmgQAAGRycy9kb3ducmV2LnhtbFBLBQYAAAAABAAEAPMAAACq
BQAAAAA=
" fillcolor="white [3201]" stroked="f" strokeweight=".5pt">
 <v:textbox>
  <![if !mso]>
  <table cellpadding=0 cellspacing=0 width="100%">
   <tr>
    <td><![endif]>
    <div>
    <p class=MsoNormal><span lang=EN-US style='color:red;mso-ansi-language:
    EN-US'>(4)<o:p></o:p></span></p>
    </div>
    <![if !mso]></td>
   </tr>
  </table>
  <![endif]></v:textbox>
</v:shape><![endif]--><![if !vml]><span style='mso-ignore:vglayout;position:
relative;z-index:251665408'><span style='left:0px;position:absolute;left:70px;
top:-42px;width:33px;height:29px'><img width=33 height=29
src="/static/RAREFAN_Manual.fld/image031.png" alt="Text Box: (4)" v:shapes="Text_x0020_Box_x0020_12"></span></span><![endif]><!--[if gte vml 1]><v:shape
 id="Text_x0020_Box_x0020_11" o:spid="_x0000_s1032" type="#_x0000_t202"
 style='position:absolute;left:0;text-align:left;margin-left:71.4pt;
 margin-top:110.5pt;width:28.4pt;height:24.55pt;text-indent:0;z-index:251663360;
 visibility:visible;mso-wrap-style:none;mso-height-percent:0;
 mso-wrap-distance-left:9pt;mso-wrap-distance-top:0;mso-wrap-distance-right:9pt;
 mso-wrap-distance-bottom:0;mso-position-horizontal:absolute;
 mso-position-horizontal-relative:text;mso-position-vertical:absolute;
 mso-position-vertical-relative:text;mso-height-percent:0;
 mso-height-relative:margin;v-text-anchor:top' o:gfxdata="UEsDBBQABgAIAAAAIQC2gziS/gAAAOEBAAATAAAAW0NvbnRlbnRfVHlwZXNdLnhtbJSRQU7DMBBF
90jcwfIWJU67QAgl6YK0S0CoHGBkTxKLZGx5TGhvj5O2G0SRWNoz/78nu9wcxkFMGNg6quQqL6RA
0s5Y6ir5vt9lD1JwBDIwOMJKHpHlpr69KfdHjyxSmriSfYz+USnWPY7AufNIadK6MEJMx9ApD/oD
OlTrorhX2lFEilmcO2RdNtjC5xDF9pCuTyYBB5bi6bQ4syoJ3g9WQ0ymaiLzg5KdCXlKLjvcW893
SUOqXwnz5DrgnHtJTxOsQfEKIT7DmDSUCaxw7Rqn8787ZsmRM9e2VmPeBN4uqYvTtW7jvijg9N/y
JsXecLq0q+WD6m8AAAD//wMAUEsDBBQABgAIAAAAIQA4/SH/1gAAAJQBAAALAAAAX3JlbHMvLnJl
bHOkkMFqwzAMhu+DvYPRfXGawxijTi+j0GvpHsDYimMaW0Yy2fr2M4PBMnrbUb/Q94l/f/hMi1qR
JVI2sOt6UJgd+ZiDgffL8ekFlFSbvV0oo4EbChzGx4f9GRdb25HMsYhqlCwG5lrLq9biZkxWOiqY
22YiTra2kYMu1l1tQD30/bPm3wwYN0x18gb45AdQl1tp5j/sFB2T0FQ7R0nTNEV3j6o9feQzro1i
OWA14Fm+Q8a1a8+Bvu/d/dMb2JY5uiPbhG/ktn4cqGU/er3pcvwCAAD//wMAUEsDBBQABgAIAAAA
IQCSy3x7PwIAAH8EAAAOAAAAZHJzL2Uyb0RvYy54bWysVMFuGjEQvVfqP1i+l10SSCnKElEiqkoo
iUSqnI3XG1byeizbsEu/vs9eIDTtqerF2DOzb2bem+H2rms02yvnazIFHw5yzpSRVNbmteA/npef
Jpz5IEwpNBlV8IPy/G728cNta6fqirakS+UYQIyftrbg2xDsNMu83KpG+AFZZeCsyDUi4Oles9KJ
FuiNzq7y/CZryZXWkVTew3rfO/ks4VeVkuGxqrwKTBcctYV0unRu4pnNbsX01Qm7reWxDPEPVTSi
Nkh6hroXQbCdq/+AamrpyFMVBpKajKqqlir1gG6G+btu1lthVeoF5Hh7psn/P1j5sH9yrC6h3ZAz
Ixpo9Ky6wL5Sx2ACP631U4StLQJDBztiT3YPY2y7q1wTf9EQgx9MH87sRjQJ45fhaJTDI+G6Hg4n
k0lEyd4+ts6Hb4oaFi8FdxAvcSr2Kx/60FNIzOVJ1+Wy1jo94sCohXZsLyC1DqlEgP8WpQ1rC35z
Pc4TsKH4eY+sDWqJrfYtxVvoNl2iZnxqd0PlASw46ufIW7msUetK+PAkHAYH7WEZwiOOShNy0fHG
2Zbcz7/ZYzz0hJezFoNYcINN4Ux/N9A5kYa5TY/R+PMVMrhLz+bSY3bNgtA+pERt6Rrjgz5dK0fN
CzZmHnPCJYxE5oKH03UR+uXAxkk1n6cgTKoVYWXWVkboSHfU4bl7Ec4exQpQ+YFOAyum7zTrY+OX
hua7QFWdBI0s95weyceUp5E4bmRco8t3inr735j9AgAA//8DAFBLAwQUAAYACAAAACEAH6xkJ+IA
AAAQAQAADwAAAGRycy9kb3ducmV2LnhtbExPO0/DMBDekfgP1iGxUScGFTeNU6GiSh1gILRidWOT
RMTnYLtt+u85JlhO993je5SryQ3sZEPsPSrIZxkwi403PbYKdu+bOwksJo1GDx6tgouNsKqur0pd
GH/GN3uqU8uIBGOhFXQpjQXnsems03HmR4u0+/TB6UQwtNwEfSZyN3CRZXPudI+k0OnRrjvbfNVH
p+B1vajlVlzCx+J+u6nld+5f5F6p25vpeUnlaQks2Sn9fcBvBvIPFRk7+COayAbCD4L8JwVC5JSM
LoSc0+RAzWOWA69K/j9I9QMAAP//AwBQSwECLQAUAAYACAAAACEAtoM4kv4AAADhAQAAEwAAAAAA
AAAAAAAAAAAAAAAAW0NvbnRlbnRfVHlwZXNdLnhtbFBLAQItABQABgAIAAAAIQA4/SH/1gAAAJQB
AAALAAAAAAAAAAAAAAAAAC8BAABfcmVscy8ucmVsc1BLAQItABQABgAIAAAAIQCSy3x7PwIAAH8E
AAAOAAAAAAAAAAAAAAAAAC4CAABkcnMvZTJvRG9jLnhtbFBLAQItABQABgAIAAAAIQAfrGQn4gAA
ABABAAAPAAAAAAAAAAAAAAAAAJkEAABkcnMvZG93bnJldi54bWxQSwUGAAAAAAQABADzAAAAqAUA
AAAA
" fillcolor="white [3201]" stroked="f" strokeweight=".5pt">
 <v:textbox>
  <![if !mso]>
  <table cellpadding=0 cellspacing=0 width="100%">
   <tr>
    <td><![endif]>
    <div>
    <p class=MsoNormal><span lang=EN-US style='color:red;mso-ansi-language:
    EN-US'>(3)<o:p></o:p></span></p>
    </div>
    <![if !mso]></td>
   </tr>
  </table>
  <![endif]></v:textbox>
</v:shape><![endif]--><![if !vml]><span style='mso-ignore:vglayout;position:
relative;z-index:251663360'><span style='left:0px;position:absolute;left:61px;
top:-78px;width:33px;height:28px'><img width=33 height=28
src="/static/RAREFAN_Manual.fld/image032.png" alt="Text Box: (3)" v:shapes="Text_x0020_Box_x0020_11"></span></span><![endif]><!--[if gte vml 1]><v:shape
 id="Text_x0020_Box_x0020_10" o:spid="_x0000_s1031" type="#_x0000_t202"
 style='position:absolute;left:0;text-align:left;margin-left:71.4pt;
 margin-top:77.55pt;width:28.4pt;height:24.55pt;text-indent:0;z-index:251661312;
 visibility:visible;mso-wrap-style:none;mso-height-percent:0;
 mso-wrap-distance-left:9pt;mso-wrap-distance-top:0;mso-wrap-distance-right:9pt;
 mso-wrap-distance-bottom:0;mso-position-horizontal:absolute;
 mso-position-horizontal-relative:text;mso-position-vertical:absolute;
 mso-position-vertical-relative:text;mso-height-percent:0;
 mso-height-relative:margin;v-text-anchor:top' o:gfxdata="UEsDBBQABgAIAAAAIQC2gziS/gAAAOEBAAATAAAAW0NvbnRlbnRfVHlwZXNdLnhtbJSRQU7DMBBF
90jcwfIWJU67QAgl6YK0S0CoHGBkTxKLZGx5TGhvj5O2G0SRWNoz/78nu9wcxkFMGNg6quQqL6RA
0s5Y6ir5vt9lD1JwBDIwOMJKHpHlpr69KfdHjyxSmriSfYz+USnWPY7AufNIadK6MEJMx9ApD/oD
OlTrorhX2lFEilmcO2RdNtjC5xDF9pCuTyYBB5bi6bQ4syoJ3g9WQ0ymaiLzg5KdCXlKLjvcW893
SUOqXwnz5DrgnHtJTxOsQfEKIT7DmDSUCaxw7Rqn8787ZsmRM9e2VmPeBN4uqYvTtW7jvijg9N/y
JsXecLq0q+WD6m8AAAD//wMAUEsDBBQABgAIAAAAIQA4/SH/1gAAAJQBAAALAAAAX3JlbHMvLnJl
bHOkkMFqwzAMhu+DvYPRfXGawxijTi+j0GvpHsDYimMaW0Yy2fr2M4PBMnrbUb/Q94l/f/hMi1qR
JVI2sOt6UJgd+ZiDgffL8ekFlFSbvV0oo4EbChzGx4f9GRdb25HMsYhqlCwG5lrLq9biZkxWOiqY
22YiTra2kYMu1l1tQD30/bPm3wwYN0x18gb45AdQl1tp5j/sFB2T0FQ7R0nTNEV3j6o9feQzro1i
OWA14Fm+Q8a1a8+Bvu/d/dMb2JY5uiPbhG/ktn4cqGU/er3pcvwCAAD//wMAUEsDBBQABgAIAAAA
IQBqE1K3PwIAAH8EAAAOAAAAZHJzL2Uyb0RvYy54bWysVMFu2zAMvQ/YPwi6L3baNMuCOkWWIsOA
oC2QDj0rstwYkEVBUmJnX78nOUmzbqdhF5kiqUfykfTtXddotlfO12QKPhzknCkjqazNa8F/PC8/
TTjzQZhSaDKq4Afl+d3s44fb1k7VFW1Jl8oxgBg/bW3BtyHYaZZ5uVWN8AOyysBYkWtEwNW9ZqUT
LdAbnV3l+ThryZXWkVTeQ3vfG/ks4VeVkuGxqrwKTBccuYV0unRu4pnNbsX01Qm7reUxDfEPWTSi
Ngh6hroXQbCdq/+AamrpyFMVBpKajKqqlirVgGqG+btq1lthVaoF5Hh7psn/P1j5sH9yrC7RO9Bj
RIMePasusK/UMajAT2v9FG5rC8fQQQ/fk95DGcvuKtfELwpisAPqcGY3okkovwxHoxwWCdP1cDiZ
TCJK9vbYOh++KWpYFAru0LzEqdivfOhdTy4xliddl8ta63SJA6MW2rG9QKt1SCkC/DcvbVhb8PH1
TZ6ADcXnPbI2yCWW2pcUpdBtukTN+FTuhsoDWHDUz5G3clkj15Xw4Uk4DA7KwzKERxyVJsSio8TZ
ltzPv+mjP/oJK2ctBrHgBpvCmf5u0OdEGuY2XUY3n68QwV1aNpcWs2sWhPKHWDorkxj9gz6JlaPm
BRszjzFhEkYicsHDSVyEfjmwcVLN58kJk2pFWJm1lRE60h378Ny9CGePzQro8gOdBlZM3/Ws940v
Dc13gao6NTSy3HN6JB9TnkbiuJFxjS7vyevtvzH7BQAA//8DAFBLAwQUAAYACAAAACEAdSkKYOMA
AAAQAQAADwAAAGRycy9kb3ducmV2LnhtbExPy07DMBC8I/EP1iJxo05MW7lpnAoVVeoBDgQQVzde
kojYDrbbpn/PcoLLamYfszPlZrIDO2GIvXcK8lkGDF3jTe9aBW+vuzsJLCbtjB68QwUXjLCprq9K
XRh/di94qlPLSMTFQivoUhoLzmPTodVx5kd0NPv0wepENLTcBH0mcTtwkWVLbnXv6EOnR9x22HzV
R6vgebuq5V5cwsfqfr+r5Xfun+S7Urc30+OaysMaWMIp/V3AbwbyDxUZO/ijM5ENxOeC/CcCi0UO
jDaEXFLnQCCbC+BVyf8HqX4AAAD//wMAUEsBAi0AFAAGAAgAAAAhALaDOJL+AAAA4QEAABMAAAAA
AAAAAAAAAAAAAAAAAFtDb250ZW50X1R5cGVzXS54bWxQSwECLQAUAAYACAAAACEAOP0h/9YAAACU
AQAACwAAAAAAAAAAAAAAAAAvAQAAX3JlbHMvLnJlbHNQSwECLQAUAAYACAAAACEAahNStz8CAAB/
BAAADgAAAAAAAAAAAAAAAAAuAgAAZHJzL2Uyb0RvYy54bWxQSwECLQAUAAYACAAAACEAdSkKYOMA
AAAQAQAADwAAAAAAAAAAAAAAAACZBAAAZHJzL2Rvd25yZXYueG1sUEsFBgAAAAAEAAQA8wAAAKkF
AAAAAA==
" fillcolor="white [3201]" stroked="f" strokeweight=".5pt">
 <v:textbox>
  <![if !mso]>
  <table cellpadding=0 cellspacing=0 width="100%">
   <tr>
    <td><![endif]>
    <div>
    <p class=MsoNormal><span lang=EN-US style='color:red;mso-ansi-language:
    EN-US'>(2)<o:p></o:p></span></p>
    </div>
    <![if !mso]></td>
   </tr>
  </table>
  <![endif]></v:textbox>
</v:shape><![endif]--><![if !vml]><span style='mso-ignore:vglayout;position:
relative;z-index:251661312'><span style='left:0px;position:absolute;left:61px;
top:-111px;width:33px;height:28px'><img width=33 height=28
src="/static/RAREFAN_Manual.fld/image033.png" alt="Text Box: (2)" v:shapes="Text_x0020_Box_x0020_10"></span></span><![endif]><!--[if gte vml 1]><v:shape
 id="Text_x0020_Box_x0020_9" o:spid="_x0000_s1030" type="#_x0000_t202" style='position:absolute;
 left:0;text-align:left;margin-left:71.45pt;margin-top:43.5pt;width:28.4pt;
 height:24.55pt;text-indent:0;z-index:251659264;visibility:visible;
 mso-wrap-style:none;mso-height-percent:0;mso-wrap-distance-left:9pt;
 mso-wrap-distance-top:0;mso-wrap-distance-right:9pt;
 mso-wrap-distance-bottom:0;mso-position-horizontal:absolute;
 mso-position-horizontal-relative:text;mso-position-vertical:absolute;
 mso-position-vertical-relative:text;mso-height-percent:0;
 mso-height-relative:margin;v-text-anchor:top' o:gfxdata="UEsDBBQABgAIAAAAIQC2gziS/gAAAOEBAAATAAAAW0NvbnRlbnRfVHlwZXNdLnhtbJSRQU7DMBBF
90jcwfIWJU67QAgl6YK0S0CoHGBkTxKLZGx5TGhvj5O2G0SRWNoz/78nu9wcxkFMGNg6quQqL6RA
0s5Y6ir5vt9lD1JwBDIwOMJKHpHlpr69KfdHjyxSmriSfYz+USnWPY7AufNIadK6MEJMx9ApD/oD
OlTrorhX2lFEilmcO2RdNtjC5xDF9pCuTyYBB5bi6bQ4syoJ3g9WQ0ymaiLzg5KdCXlKLjvcW893
SUOqXwnz5DrgnHtJTxOsQfEKIT7DmDSUCaxw7Rqn8787ZsmRM9e2VmPeBN4uqYvTtW7jvijg9N/y
JsXecLq0q+WD6m8AAAD//wMAUEsDBBQABgAIAAAAIQA4/SH/1gAAAJQBAAALAAAAX3JlbHMvLnJl
bHOkkMFqwzAMhu+DvYPRfXGawxijTi+j0GvpHsDYimMaW0Yy2fr2M4PBMnrbUb/Q94l/f/hMi1qR
JVI2sOt6UJgd+ZiDgffL8ekFlFSbvV0oo4EbChzGx4f9GRdb25HMsYhqlCwG5lrLq9biZkxWOiqY
22YiTra2kYMu1l1tQD30/bPm3wwYN0x18gb45AdQl1tp5j/sFB2T0FQ7R0nTNEV3j6o9feQzro1i
OWA14Fm+Q8a1a8+Bvu/d/dMb2JY5uiPbhG/ktn4cqGU/er3pcvwCAAD//wMAUEsDBBQABgAIAAAA
IQB+lKLxQAIAAH0EAAAOAAAAZHJzL2Uyb0RvYy54bWysVMFu2zAMvQ/YPwi6L7bTtE2MOEWWIsOA
oi2QDD0rshwLkEVBUmJnXz9KjtOs22nYRaZI6pF8JD1/6BpFjsI6Cbqg2SilRGgOpdT7gv7Yrr9M
KXGe6ZIp0KKgJ+How+Lzp3lrcjGGGlQpLEEQ7fLWFLT23uRJ4ngtGuZGYIRGYwW2YR6vdp+UlrWI
3qhknKZ3SQu2NBa4cA61j72RLiJ+VQnuX6rKCU9UQTE3H08bz104k8Wc5XvLTC35OQ32D1k0TGoM
eoF6ZJ6Rg5V/QDWSW3BQ+RGHJoGqklzEGrCaLP1QzaZmRsRakBxnLjS5/wfLn4+vlsiyoDNKNGuw
RVvRefIVOjIL7LTG5ei0MejmO1Rjlwe9Q2UouqtsE75YDkE78ny6cBvAOCpn2WSSooWj6SbLptNp
QEneHxvr/DcBDQlCQS22LjLKjk/O966DS4jlQMlyLZWKlzAuYqUsOTJstPIxRQT/zUtp0hb07uY2
jcAawvMeWWnMJZTalxQk3+26SMz9UO4OyhOyYKGfImf4WmKuT8z5V2ZxbLA8XAX/gkelAGPBWaKk
Bvvzb/rgj91EKyUtjmFBNe4JJeq7xi5H0nBq42Vyez/GCPbasru26EOzAiw/w5UzPIrB36tBrCw0
b7gvyxATTUxzjFxQP4gr368G7hsXy2V0wjk1zD/pjeEBOtAd+rDt3pg152Z57PIzDOPK8g89633D
Sw3Lg4dKxoYGlntOz+TjjMeROO9jWKLre/R6/2ssfgEAAP//AwBQSwMEFAAGAAgAAAAhALBjeFLj
AAAADwEAAA8AAABkcnMvZG93bnJldi54bWxMT0FOwzAQvCPxB2uRuFEnKQpOGqdCRZV6gEMDiKsb
L0lEbAfbbdPfs5zgstLszM7OVOvZjOyEPgzOSkgXCTC0rdOD7SS8vW7vBLAQldVqdBYlXDDAur6+
qlSp3dnu8dTEjpGJDaWS0Mc4lZyHtkejwsJNaIn7dN6oSNB3XHt1JnMz8ixJcm7UYOlDrybc9Nh+
NUcj4WVTNGKXXfxHsdxtG/GdumfxLuXtzfy0ovG4AhZxjn8X8NuB8kNNwQ7uaHVgI+H7rCCpBPFA
xUiQiZwWB2KWeQq8rvj/HvUPAAAA//8DAFBLAQItABQABgAIAAAAIQC2gziS/gAAAOEBAAATAAAA
AAAAAAAAAAAAAAAAAABbQ29udGVudF9UeXBlc10ueG1sUEsBAi0AFAAGAAgAAAAhADj9If/WAAAA
lAEAAAsAAAAAAAAAAAAAAAAALwEAAF9yZWxzLy5yZWxzUEsBAi0AFAAGAAgAAAAhAH6UovFAAgAA
fQQAAA4AAAAAAAAAAAAAAAAALgIAAGRycy9lMm9Eb2MueG1sUEsBAi0AFAAGAAgAAAAhALBjeFLj
AAAADwEAAA8AAAAAAAAAAAAAAAAAmgQAAGRycy9kb3ducmV2LnhtbFBLBQYAAAAABAAEAPMAAACq
BQAAAAA=
" fillcolor="white [3201]" stroked="f" strokeweight=".5pt">
 <v:textbox>
  <![if !mso]>
  <table cellpadding=0 cellspacing=0 width="100%">
   <tr>
    <td><![endif]>
    <div>
    <p class=MsoNormal><span lang=EN-US style='color:red;mso-ansi-language:
    EN-US'>(1)<o:p></o:p></span></p>
    </div>
    <![if !mso]></td>
   </tr>
  </table>
  <![endif]></v:textbox>
</v:shape><![endif]--><![if !vml]><span style='mso-ignore:vglayout;position:
relative;z-index:251659264'><span style='left:0px;position:absolute;left:61px;
top:-145px;width:33px;height:28px'><img width=33 height=28
src="/static/RAREFAN_Manual.fld/image034.png" alt="Text Box: (1)" v:shapes="Text_x0020_Box_x0020_9"></span></span><![endif]><!--[if gte vml 1]><v:shape
 id="Picture_x0020_2" o:spid="_x0000_s1029" type="#_x0000_t75" style='position:absolute;
 left:0;text-align:left;margin-left:0;margin-top:0;width:263.4pt;height:329.85pt;
 z-index:251658239;visibility:visible;mso-wrap-style:square;
 mso-wrap-distance-left:9pt;mso-wrap-distance-top:0;mso-wrap-distance-right:9pt;
 mso-wrap-distance-bottom:0;mso-position-horizontal:left;
 mso-position-horizontal-relative:margin;mso-position-vertical:top;
 mso-position-vertical-relative:margin' stroked="t" strokecolor="black [3213]">
 <v:imagedata src="/static/RAREFAN_Manual.fld/image011.png" o:title=""/>
 <w:wrap type="square" anchorx="margin" anchory="margin"/>
</v:shape><![endif]--><![if !vml]><img width=263 height=330
src="/static/RAREFAN_Manual.fld/image035.png" align=left hspace=12 v:shapes="Picture_x0020_2"><![endif]><span
lang=EN-US style='mso-ansi-language:EN-US'>Please select a reference sequence
from the sequence files you submitted. The reference sequence will be used to
identify a maximum of six seed sequences. The seed sequences have to occur at
least n times (specified in <span style='color:red'>(3)</span>) and are of
length N (specified in <span style='color:red'>(5)</span>) and are the basis
for determining sequence groups. The way sequence groups are created is
described in detail here </span><!--[if supportFields]><span lang=EN-US
style='mso-ansi-language:EN-US'><span style='mso-element:field-begin'></span><span
style='mso-spacerun:yes'> </span>ADDIN ZOTERO_ITEM CSL_CITATION
{&quot;citationID&quot;:&quot;dKpUkMV8&quot;,&quot;properties&quot;:{&quot;formattedCitation&quot;:&quot;(3)&quot;,&quot;plainCitation&quot;:&quot;(3)&quot;,&quot;noteIndex&quot;:0},&quot;citationItems&quot;:[{&quot;id&quot;:5,&quot;uris&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/CJYAX3YY&quot;],&quot;uri&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/CJYAX3YY&quot;],&quot;itemData&quot;:{&quot;id&quot;:5,&quot;type&quot;:&quot;article-journal&quot;,&quot;abstract&quot;:&quot;Author
Summary DNA sequences that copy themselves throughout genomes, and make no
specific contribution to reproductive success, are by definition â€œselfish.â€
Such DNA is a feature of the genomes of all organisms and evident by virtue of
its repetitive nature. In bacteria the predominant repetitive sequences are
short (âˆ¼20 bp), extragenic, and palindromic. These so-called REP sequences
may occur many hundreds of times per genome, but their origins and means of
dissemination have been a longstanding mystery. We show that REPs are
components of higher-order replicative entities termed REPINs, which are
themselves thought to be derived from REP sequences that flanked an ancestral
autonomous selfish element. In this ancestral state the REP sequences were
likely to have been critical for the movement of the selfish element, but were
devoid of any capacity to replicate independently. REPINs, on the other hand,
have evolved to have a life of their own, albeit one that exploitsâ€”even
enslavesâ€”a genetic element upon which their existence depends. REPINs are the
ultimate non-autonomous, super-streamlined, selfish element and are widespread
among bacteria.&quot;,&quot;container-title&quot;:&quot;PLoS
genetics&quot;,&quot;DOI&quot;:&quot;10.1371/journal.pgen.1002132&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;journalAbbreviation&quot;:&quot;PLoS
Genet.&quot;,&quot;note&quot;:&quot;publisher: Public Library of Science&quot;,&quot;page&quot;:&quot;e1002132&quot;,&quot;title&quot;:&quot;Within-Genome
Evolution of REPINs: a New Family of Miniature Mobile DNA in
Bacteria&quot;,&quot;volume&quot;:&quot;7&quot;,&quot;author&quot;:[{&quot;family&quot;:&quot;Bertels&quot;,&quot;given&quot;:&quot;Frederic&quot;},{&quot;family&quot;:&quot;Rainey&quot;,&quot;given&quot;:&quot;Paul
B&quot;}],&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2011&quot;,6,1]]}}}],&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}
<span style='mso-element:field-separator'></span></span><![endif]--><span
lang=EN-US style='mso-ansi-language:EN-US'><span style='mso-no-proof:yes'>(3)</span></span><!--[if supportFields]><span
lang=EN-US style='mso-ansi-language:EN-US'><span style='mso-element:field-end'></span></span><![endif]--><span
lang=EN-US style='mso-ansi-language:EN-US'>, in the section “Grouping of highly
abundant oligonucleotides in SBW25”. If sequence groups are part of REPINs,
then each sequence group can be used to define a REPIN type.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoListParagraphCxSpFirst style='margin-left:0cm;mso-add-space:auto;
text-indent:-18.0pt;mso-list:l0 level1 lfo1;tab-stops:290.6pt'><![if !supportLists]><span
lang=EN-US style='color:red;mso-ansi-language:EN-US'><span style='mso-list:
Ignore'>(2)<span style='font:7.0pt "Times New Roman"'>&nbsp; </span></span></span><![endif]><span
lang=EN-US style='mso-ansi-language:EN-US'>There are two very divergent RAYT
groups (<i>E. coli </i><span style='mso-spacerun:yes'> </span>group and <i>P.
fluorescens </i>group). If you do not know what kind of RAYT is present in your
genome then you may have to run RAREFAN twice, once with the SBW25 and once
with the <i>E. coli </i>RAYT. If you have supplied a RAYT protein yourself you
should be able to select it here. These sequences are then used as query
proteins in a TBLASTN </span><!--[if supportFields]><span lang=EN-US
style='mso-ansi-language:EN-US'><span style='mso-element:field-begin'></span><span
style='mso-spacerun:yes'> </span>ADDIN ZOTERO_ITEM CSL_CITATION
{&quot;citationID&quot;:&quot;c035UzBk&quot;,&quot;properties&quot;:{&quot;formattedCitation&quot;:&quot;(4)&quot;,&quot;plainCitation&quot;:&quot;(4)&quot;,&quot;noteIndex&quot;:0},&quot;citationItems&quot;:[{&quot;id&quot;:258,&quot;uris&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/D2SE3MXG&quot;],&quot;uri&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/D2SE3MXG&quot;],&quot;itemData&quot;:{&quot;id&quot;:258,&quot;type&quot;:&quot;article-journal&quot;,&quot;abstract&quot;:&quot;BACKGROUND:Sequence
similarity searching is a very important bioinformatics task. While Basic Local
Alignment Search Tool (BLAST) outperforms exact methods through its use of
heuristics, the speed of the current BLAST software is suboptimal for very long
queries or database sequences. There are also some shortcomings in the
user-interface of the current command-line applications.\n\nRESULTS:We describe
features and improvements of rewritten BLAST software and introduce new
command-line applications. Long query sequences are broken into chunks for
processing, in some cases leading to dramatically shorter run times. For long
database sequences, it is possible to retrieve only the relevant parts of the
sequence, reducing CPU time and memory usage for searches of short queries against
databases of contigs or chromosomes. The program can now retrieve masking
information for database sequences from the BLAST databases. A new modular
software library can now access subject sequence data from arbitrary data
sources. We introduce several new features, including strategy files that allow
a user to save and reuse their favorite set of options. The strategy files can
be uploaded to and downloaded from the NCBI BLAST web site.\n\nCONCLUSION:The
new BLAST command-line applications, compared to the current BLAST tools,
demonstrate substantial speed improvements for long queries as well as
chromosome length database sequences. We have also improved the user interface
of the command-line applications.&quot;,&quot;container-title&quot;:&quot;BMC
Bioinformatics&quot;,&quot;DOI&quot;:&quot;10.1186/1471-2105-10-421&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;journalAbbreviation&quot;:&quot;BMC
Bioinformatics&quot;,&quot;note&quot;:&quot;publisher: BioMed
Central&quot;,&quot;page&quot;:&quot;421-9&quot;,&quot;title&quot;:&quot;BLAST+:
architecture and
applications.&quot;,&quot;volume&quot;:&quot;10&quot;,&quot;author&quot;:[{&quot;family&quot;:&quot;Camacho&quot;,&quot;given&quot;:&quot;Christiam&quot;},{&quot;family&quot;:&quot;Coulouris&quot;,&quot;given&quot;:&quot;George&quot;},{&quot;family&quot;:&quot;Avagyan&quot;,&quot;given&quot;:&quot;Vahram&quot;},{&quot;family&quot;:&quot;Ma&quot;,&quot;given&quot;:&quot;Ning&quot;},{&quot;family&quot;:&quot;Papadopoulos&quot;,&quot;given&quot;:&quot;Jason&quot;},{&quot;family&quot;:&quot;Bealer&quot;,&quot;given&quot;:&quot;Kevin&quot;},{&quot;family&quot;:&quot;Madden&quot;,&quot;given&quot;:&quot;Thomas
L&quot;}],&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2009&quot;,12,15]]}}}],&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}
<span style='mso-element:field-separator'></span></span><![endif]--><span
lang=EN-US style='mso-ansi-language:EN-US'><span style='mso-no-proof:yes'>(4)</span></span><!--[if supportFields]><span
lang=EN-US style='mso-ansi-language:EN-US'><span style='mso-element:field-end'></span></span><![endif]--><span
lang=EN-US style='mso-ansi-language:EN-US'> search of the provided genomes. All
sequences that are identified below the e-Value threshold set in <span
style='color:red'>(6)</span><span style='color:black;mso-themecolor:text1'>.</span><o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US style='mso-ansi-language:
EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0cm;mso-add-space:auto;
text-indent:0cm;mso-list:l0 level1 lfo1;tab-stops:21.3pt'><![if !supportLists]><span
lang=EN-US style='color:red;mso-ansi-language:EN-US'><span style='mso-list:
Ignore'>(3)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
lang=EN-US style='mso-ansi-language:EN-US'>If you have provided a tree file in
the previous step then you can select it here.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US style='mso-ansi-language:
EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0cm;mso-add-space:auto;
text-indent:0cm;mso-list:l0 level1 lfo1;tab-stops:21.3pt'><![if !supportLists]><span
lang=EN-US style='color:red;mso-ansi-language:EN-US'><span style='mso-list:
Ignore'>(4)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
lang=EN-US style='mso-ansi-language:EN-US'>This parameter is required for
identifying REPINs. Only sequences that occur more frequently than this value
will be considered when identifying REPINs. <o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US style='mso-ansi-language:
EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0cm;mso-add-space:auto;
text-indent:0cm;mso-list:l0 level1 lfo1;tab-stops:21.3pt'><![if !supportLists]><span
lang=EN-US style='color:red;mso-ansi-language:EN-US'><span style='mso-list:
Ignore'>(5)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
lang=EN-US style='mso-ansi-language:EN-US'>The seed sequence length for
identifying REPINs. <o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US style='mso-ansi-language:
EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0cm;mso-add-space:auto;
text-indent:0cm;mso-list:l0 level1 lfo1;tab-stops:21.3pt'><![if !supportLists]><span
lang=EN-US style='color:red;mso-ansi-language:EN-US'><span style='mso-list:
Ignore'>(6)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
lang=EN-US style='mso-ansi-language:EN-US'>This e-Value determines, which genes
are identified as RAYTs in the genome.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US style='mso-ansi-language:
EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:0cm;mso-add-space:auto;
text-indent:0cm;mso-list:l0 level1 lfo1;tab-stops:21.3pt'><![if !supportLists]><span
lang=EN-US style='color:red;mso-ansi-language:EN-US'><span style='mso-list:
Ignore'>(7)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
lang=EN-US style='mso-ansi-language:EN-US'>If this box is ticked then REPINs
are identified. REPINs are defined as sequences that consist of two seed
sequences in inverted orientation that are found at a distance of less than 130
bp. If the box is unticked then the seed sequences (REPs) identified as
described in <span style='color:red'>(1)</span> are used for all further
analyses. This can be useful if REPINs are asymmetric (i.e. there is for
example a deletion/insertion in either the 5’ or 3’ REP sequence). This is for
example the case for <i>E. coli </i>REPINs.<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle><span lang=EN-US style='mso-ansi-language:
EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoListParagraphCxSpLast style='margin-left:0cm;mso-add-space:auto;
text-indent:0cm;mso-list:l0 level1 lfo1;tab-stops:21.3pt'><![if !supportLists]><span
lang=EN-US style='color:red;mso-ansi-language:EN-US'><span style='mso-list:
Ignore'>(8)<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
lang=EN-US style='mso-ansi-language:EN-US'>If you provide an email address here
you will be notified once the job is finished. This is particular useful when
applying RAREFAN to large datasets.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US;mso-no-proof:
yes'><!--[if gte vml 1]><v:shape id="Picture_x0020_3" o:spid="_x0000_i1030"
 type="#_x0000_t75" style='width:283pt;height:84pt;visibility:visible;
 mso-wrap-style:square' o:bordertopcolor="yellow pure" o:borderleftcolor="yellow pure"
 o:borderbottomcolor="yellow pure" o:borderrightcolor="yellow pure">
 <v:imagedata src="/static/RAREFAN_Manual.fld/image013.png" o:title=""/>
 <w:bordertop type="single" width="6"/>
 <w:borderleft type="single" width="6"/>
 <w:borderbottom type="single" width="6"/>
 <w:borderright type="single" width="6"/>
</v:shape><![endif]--><![if !vml]><img width=285 height=86
src="/static/RAREFAN_Manual.fld/image036.png" v:shapes="Picture_x0020_3"><![endif]></span><span
lang=EN-US style='mso-ansi-language:EN-US'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'>Once the
job is submit, it is assigned a unique identifier. If you have not supplied an
email address you will either need to keep the site open or remember the unique
ID to access the website later. If you have closed the website but know the
unique ID you can access your data by calling this address http://rarefan.evolbio.mpg.de/results?run_id=<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><!--[if gte vml 1]><v:shape id="Text_x0020_Box_x0020_18"
 o:spid="_x0000_s1028" type="#_x0000_t202" style='position:absolute;
 margin-left:45.7pt;margin-top:86.3pt;width:28.4pt;height:21.2pt;z-index:251674624;
 visibility:visible;mso-wrap-style:none;mso-height-percent:0;
 mso-wrap-distance-left:9pt;mso-wrap-distance-top:0;mso-wrap-distance-right:9pt;
 mso-wrap-distance-bottom:0;mso-position-horizontal:absolute;
 mso-position-horizontal-relative:text;mso-position-vertical:absolute;
 mso-position-vertical-relative:text;mso-height-percent:0;
 mso-height-relative:margin;v-text-anchor:top' o:gfxdata="UEsDBBQABgAIAAAAIQC2gziS/gAAAOEBAAATAAAAW0NvbnRlbnRfVHlwZXNdLnhtbJSRQU7DMBBF
90jcwfIWJU67QAgl6YK0S0CoHGBkTxKLZGx5TGhvj5O2G0SRWNoz/78nu9wcxkFMGNg6quQqL6RA
0s5Y6ir5vt9lD1JwBDIwOMJKHpHlpr69KfdHjyxSmriSfYz+USnWPY7AufNIadK6MEJMx9ApD/oD
OlTrorhX2lFEilmcO2RdNtjC5xDF9pCuTyYBB5bi6bQ4syoJ3g9WQ0ymaiLzg5KdCXlKLjvcW893
SUOqXwnz5DrgnHtJTxOsQfEKIT7DmDSUCaxw7Rqn8787ZsmRM9e2VmPeBN4uqYvTtW7jvijg9N/y
JsXecLq0q+WD6m8AAAD//wMAUEsDBBQABgAIAAAAIQA4/SH/1gAAAJQBAAALAAAAX3JlbHMvLnJl
bHOkkMFqwzAMhu+DvYPRfXGawxijTi+j0GvpHsDYimMaW0Yy2fr2M4PBMnrbUb/Q94l/f/hMi1qR
JVI2sOt6UJgd+ZiDgffL8ekFlFSbvV0oo4EbChzGx4f9GRdb25HMsYhqlCwG5lrLq9biZkxWOiqY
22YiTra2kYMu1l1tQD30/bPm3wwYN0x18gb45AdQl1tp5j/sFB2T0FQ7R0nTNEV3j6o9feQzro1i
OWA14Fm+Q8a1a8+Bvu/d/dMb2JY5uiPbhG/ktn4cqGU/er3pcvwCAAD//wMAUEsDBBQABgAIAAAA
IQAJRlhmPwIAAH8EAAAOAAAAZHJzL2Uyb0RvYy54bWysVN9v2jAQfp+0/8Hy+0igha2ooWJUTJNQ
W6lMfTaOUyI5Pss2JOyv32cHKOv2NO3FOd+d78f33eX2rms02yvnazIFHw5yzpSRVNbmteA/1stP
XzjzQZhSaDKq4Afl+d3s44fb1k7ViLakS+UYghg/bW3BtyHYaZZ5uVWN8AOyysBYkWtEwNW9ZqUT
LaI3Ohvl+SRryZXWkVTeQ3vfG/ksxa8qJcNjVXkVmC44agvpdOncxDOb3YrpqxN2W8tjGeIfqmhE
bZD0HOpeBMF2rv4jVFNLR56qMJDUZFRVtVSpB3QzzN9187wVVqVeAI63Z5j8/wsrH/ZPjtUluANT
RjTgaK26wL5Sx6ACPq31U7g9WziGDnr4nvQeyth2V7kmftEQgx1IH87oxmgSypvh9XUOi4RpNLm5
Gqfo2dtj63z4pqhhUSi4A3kJU7Ff+YBC4Hpyibk86bpc1lqnSxwYtdCO7QWo1iGViBe/eWnD2oJP
rsZ5CmwoPu8ja4MEsdW+pSiFbtMlaM4wbKg8AAVH/Rx5K5c1al0JH56Ew+CgPSxDeMRRaUIuOkqc
bcn9/Js++oNPWDlrMYgFN9gUzvR3A54TaJjbdLkefx4hg7u0bC4tZtcsCO0PsXRWJjH6B30SK0fN
CzZmHnPCJIxE5oKHk7gI/XJg46Saz5MTJtWKsDLPVsbQEe7Iw7p7Ec4eyQpg+YFOAyum7zjrfeNL
Q/NdoKpOhEaUe0yP4GPKE8/HjYxrdHlPXm//jdkvAAAA//8DAFBLAwQUAAYACAAAACEAMvOYyeQA
AAAPAQAADwAAAGRycy9kb3ducmV2LnhtbExPwU7DMAy9I/EPkZG4sbQdG23XdEJDk3ZgBwqIa9aY
tqJxSpNt3d9jTnCx7Ofn5/eK9WR7ccLRd44UxLMIBFLtTEeNgrfX7V0KwgdNRveOUMEFPazL66tC
58ad6QVPVWgEi5DPtYI2hCGX0tctWu1nbkDi3acbrQ48jo00oz6zuO1lEkVLaXVH/KHVA25arL+q
o1Ww32RVuksu40c2322r9Dt2z+m7Urc309OKy+MKRMAp/F3Abwb2DyUbO7gjGS96BVl8z0zGH5Il
CCYk8wUjB27iRQSyLOT/HOUPAAAA//8DAFBLAQItABQABgAIAAAAIQC2gziS/gAAAOEBAAATAAAA
AAAAAAAAAAAAAAAAAABbQ29udGVudF9UeXBlc10ueG1sUEsBAi0AFAAGAAgAAAAhADj9If/WAAAA
lAEAAAsAAAAAAAAAAAAAAAAALwEAAF9yZWxzLy5yZWxzUEsBAi0AFAAGAAgAAAAhAAlGWGY/AgAA
fwQAAA4AAAAAAAAAAAAAAAAALgIAAGRycy9lMm9Eb2MueG1sUEsBAi0AFAAGAAgAAAAhADLzmMnk
AAAADwEAAA8AAAAAAAAAAAAAAAAAmQQAAGRycy9kb3ducmV2LnhtbFBLBQYAAAAABAAEAPMAAACq
BQAAAAA=
" fillcolor="white [3201]" stroked="f" strokeweight=".5pt">
 <v:textbox>
  <![if !mso]>
  <table cellpadding=0 cellspacing=0 width="100%">
   <tr>
    <td><![endif]>
    <div>
    <p class=MsoNormal><span lang=EN-US style='color:red;mso-ansi-language:
    EN-US'>(1)<o:p></o:p></span></p>
    </div>
    <![if !mso]></td>
   </tr>
  </table>
  <![endif]></v:textbox>
</v:shape><![endif]--><![if !vml]><span style='mso-ignore:vglayout;position:
relative;z-index:251674624'><span style='position:absolute;left:36px;
top:-398px;width:32px;height:26px'><img width=32 height=26
src="/static/RAREFAN_Manual.fld/image037.png" alt="Text Box: (1)" v:shapes="Text_x0020_Box_x0020_18"></span></span><![endif]><!--[if gte vml 1]><v:shape
 id="Text_x0020_Box_x0020_20" o:spid="_x0000_s1027" type="#_x0000_t202"
 style='position:absolute;margin-left:52.45pt;margin-top:145.55pt;width:28.4pt;
 height:21.2pt;z-index:251678720;visibility:visible;mso-wrap-style:none;
 mso-height-percent:0;mso-wrap-distance-left:9pt;mso-wrap-distance-top:0;
 mso-wrap-distance-right:9pt;mso-wrap-distance-bottom:0;
 mso-position-horizontal:absolute;mso-position-horizontal-relative:text;
 mso-position-vertical:absolute;mso-position-vertical-relative:text;
 mso-height-percent:0;mso-height-relative:margin;v-text-anchor:top' o:gfxdata="UEsDBBQABgAIAAAAIQC2gziS/gAAAOEBAAATAAAAW0NvbnRlbnRfVHlwZXNdLnhtbJSRQU7DMBBF
90jcwfIWJU67QAgl6YK0S0CoHGBkTxKLZGx5TGhvj5O2G0SRWNoz/78nu9wcxkFMGNg6quQqL6RA
0s5Y6ir5vt9lD1JwBDIwOMJKHpHlpr69KfdHjyxSmriSfYz+USnWPY7AufNIadK6MEJMx9ApD/oD
OlTrorhX2lFEilmcO2RdNtjC5xDF9pCuTyYBB5bi6bQ4syoJ3g9WQ0ymaiLzg5KdCXlKLjvcW893
SUOqXwnz5DrgnHtJTxOsQfEKIT7DmDSUCaxw7Rqn8787ZsmRM9e2VmPeBN4uqYvTtW7jvijg9N/y
JsXecLq0q+WD6m8AAAD//wMAUEsDBBQABgAIAAAAIQA4/SH/1gAAAJQBAAALAAAAX3JlbHMvLnJl
bHOkkMFqwzAMhu+DvYPRfXGawxijTi+j0GvpHsDYimMaW0Yy2fr2M4PBMnrbUb/Q94l/f/hMi1qR
JVI2sOt6UJgd+ZiDgffL8ekFlFSbvV0oo4EbChzGx4f9GRdb25HMsYhqlCwG5lrLq9biZkxWOiqY
22YiTra2kYMu1l1tQD30/bPm3wwYN0x18gb45AdQl1tp5j/sFB2T0FQ7R0nTNEV3j6o9feQzro1i
OWA14Fm+Q8a1a8+Bvu/d/dMb2JY5uiPbhG/ktn4cqGU/er3pcvwCAAD//wMAUEsDBBQABgAIAAAA
IQDrc+P2PwIAAH8EAAAOAAAAZHJzL2Uyb0RvYy54bWysVMFuGjEQvVfqP1i+lwUSaEBZIkpEVQkl
kaDK2Xi9YSWvx7INu/Tr++yFhKQ9Vb2Y8czs88x7M9zetbVmB+V8RSbng16fM2UkFZV5yfnPzfLL
DWc+CFMITUbl/Kg8v5t9/nTb2Kka0o50oRwDiPHTxuZ8F4KdZpmXO1UL3yOrDIIluVoEXN1LVjjR
AL3W2bDfH2cNucI6ksp7eO+7IJ8l/LJUMjyWpVeB6ZyjtpBOl85tPLPZrZi+OGF3lTyVIf6hilpU
Bo++Qt2LINjeVX9A1ZV05KkMPUl1RmVZSZV6QDeD/odu1jthVeoF5Hj7SpP/f7Dy4fDkWFXkfAh6
jKih0Ua1gX2jlsEFfhrrp0hbWySGFn7ofPZ7OGPbbenq+IuGGOKAOr6yG9EknJPB9XUfEYnQcDy5
Gt1ElOztY+t8+K6oZtHIuYN4iVNxWPnQpZ5T4luedFUsK63TJQ6MWmjHDgJS65BKBPi7LG1Yk/Px
1aifgA3FzztkbVBLbLVrKVqh3baJmsm53S0VR7DgqJsjb+WyQq0r4cOTcBgctIdlCI84Sk14i04W
Zztyv/7mj/nQE1HOGgxizg02hTP9w0DnRBrmNl2uR1+jSu4ysr2MmH29ILQ/wNJZmcyYH/TZLB3V
z9iYeXwTIWEkXs55OJuL0C0HNk6q+TwlYVKtCCuztjJCR7qjDpv2WTh7EitA5Qc6D6yYftCsy41f
GprvA5VVEjSy3HF6Ih9TnkbitJFxjS7vKevtf2P2GwAA//8DAFBLAwQUAAYACAAAACEA2L9Q/OQA
AAAQAQAADwAAAGRycy9kb3ducmV2LnhtbExPy07DMBC8I/EP1iJxo86jICeNU6GiSj3AoQHE1Y2X
JCK2g+226d+znOCy0uzOzqNaz2ZkJ/RhcFZCukiAoW2dHmwn4e11eyeAhaisVqOzKOGCAdb19VWl
Su3Odo+nJnaMRGwolYQ+xqnkPLQ9GhUWbkJLt0/njYoEfce1V2cSNyPPkuSBGzVYcujVhJse26/m
aCS8bIpG7LKL/yjy3bYR36l7Fu9S3t7MTysajytgEef49wG/HSg/1BTs4I5WBzYSTpYFUSVkRZoC
I0a2FLQ5SMjz/B54XfH/ReofAAAA//8DAFBLAQItABQABgAIAAAAIQC2gziS/gAAAOEBAAATAAAA
AAAAAAAAAAAAAAAAAABbQ29udGVudF9UeXBlc10ueG1sUEsBAi0AFAAGAAgAAAAhADj9If/WAAAA
lAEAAAsAAAAAAAAAAAAAAAAALwEAAF9yZWxzLy5yZWxzUEsBAi0AFAAGAAgAAAAhAOtz4/Y/AgAA
fwQAAA4AAAAAAAAAAAAAAAAALgIAAGRycy9lMm9Eb2MueG1sUEsBAi0AFAAGAAgAAAAhANi/UPzk
AAAAEAEAAA8AAAAAAAAAAAAAAAAAmQQAAGRycy9kb3ducmV2LnhtbFBLBQYAAAAABAAEAPMAAACq
BQAAAAA=
" fillcolor="white [3201]" stroked="f" strokeweight=".5pt">
 <v:textbox>
  <![if !mso]>
  <table cellpadding=0 cellspacing=0 width="100%">
   <tr>
    <td><![endif]>
    <div>
    <p class=MsoNormal><span lang=EN-US style='color:red;mso-ansi-language:
    EN-US'>(3)<o:p></o:p></span></p>
    </div>
    <![if !mso]></td>
   </tr>
  </table>
  <![endif]></v:textbox>
</v:shape><![endif]--><![if !vml]><span style='mso-ignore:vglayout;position:
relative;z-index:251678720'><span style='position:absolute;left:42px;
top:-338px;width:33px;height:25px'><img width=33 height=25
src="/static/RAREFAN_Manual.fld/image038.png" alt="Text Box: (3)" v:shapes="Text_x0020_Box_x0020_20"></span></span><![endif]><!--[if gte vml 1]><v:shape
 id="Text_x0020_Box_x0020_19" o:spid="_x0000_s1026" type="#_x0000_t202"
 style='position:absolute;margin-left:69.75pt;margin-top:127.6pt;width:28.4pt;
 height:21.2pt;z-index:251676672;visibility:visible;mso-wrap-style:none;
 mso-height-percent:0;mso-wrap-distance-left:9pt;mso-wrap-distance-top:0;
 mso-wrap-distance-right:9pt;mso-wrap-distance-bottom:0;
 mso-position-horizontal:absolute;mso-position-horizontal-relative:text;
 mso-position-vertical:absolute;mso-position-vertical-relative:text;
 mso-height-percent:0;mso-height-relative:margin;v-text-anchor:top' o:gfxdata="UEsDBBQABgAIAAAAIQC2gziS/gAAAOEBAAATAAAAW0NvbnRlbnRfVHlwZXNdLnhtbJSRQU7DMBBF
90jcwfIWJU67QAgl6YK0S0CoHGBkTxKLZGx5TGhvj5O2G0SRWNoz/78nu9wcxkFMGNg6quQqL6RA
0s5Y6ir5vt9lD1JwBDIwOMJKHpHlpr69KfdHjyxSmriSfYz+USnWPY7AufNIadK6MEJMx9ApD/oD
OlTrorhX2lFEilmcO2RdNtjC5xDF9pCuTyYBB5bi6bQ4syoJ3g9WQ0ymaiLzg5KdCXlKLjvcW893
SUOqXwnz5DrgnHtJTxOsQfEKIT7DmDSUCaxw7Rqn8787ZsmRM9e2VmPeBN4uqYvTtW7jvijg9N/y
JsXecLq0q+WD6m8AAAD//wMAUEsDBBQABgAIAAAAIQA4/SH/1gAAAJQBAAALAAAAX3JlbHMvLnJl
bHOkkMFqwzAMhu+DvYPRfXGawxijTi+j0GvpHsDYimMaW0Yy2fr2M4PBMnrbUb/Q94l/f/hMi1qR
JVI2sOt6UJgd+ZiDgffL8ekFlFSbvV0oo4EbChzGx4f9GRdb25HMsYhqlCwG5lrLq9biZkxWOiqY
22YiTra2kYMu1l1tQD30/bPm3wwYN0x18gb45AdQl1tp5j/sFB2T0FQ7R0nTNEV3j6o9feQzro1i
OWA14Fm+Q8a1a8+Bvu/d/dMb2JY5uiPbhG/ktn4cqGU/er3pcvwCAAD//wMAUEsDBBQABgAIAAAA
IQBYdMDzPwIAAIAEAAAOAAAAZHJzL2Uyb0RvYy54bWysVE1v2zAMvQ/YfxB0X5ykadcEcYosRYYB
RVsgGXpWZDkxIIuCpMbufv2e5Hy122nYRaZI6pF8JD29a2vN9sr5ikzOB70+Z8pIKiqzzfnP9fLL
LWc+CFMITUbl/E15fjf7/Gna2Ika0o50oRwDiPGTxuZ8F4KdZJmXO1UL3yOrDIwluVoEXN02K5xo
gF7rbNjv32QNucI6ksp7aO87I58l/LJUMjyVpVeB6Zwjt5BOl85NPLPZVEy2TthdJQ9piH/IohaV
QdAT1L0Igr266g+oupKOPJWhJ6nOqCwrqVINqGbQ/1DNaiesSrWAHG9PNPn/Bysf98+OVQV6N+bM
iBo9Wqs2sG/UMqjAT2P9BG4rC8fQQg/fo95DGctuS1fHLwpisIPptxO7EU1COR6MRn1YJEzDm/HV
9W1Eyc6PrfPhu6KaRSHnDs1LnIr9gw+d69ElxvKkq2JZaZ0ucWDUQju2F2i1DilFgL/z0oY1Ob+5
uu4nYEPxeYesDXKJpXYlRSm0m7ajJs1JVG2oeAMNjrpB8lYuKyT7IHx4Fg6Tg/qwDeEJR6kJwegg
cbYj9+tv+uiPhsLKWYNJzLnBqnCmfxg0OrGGwU2X0fXXISK4S8vm0mJe6wWh/gG2zsokRv+gj2Lp
qH7BysxjTJiEkYic83AUF6HbDqycVPN5csKoWhEezMrKCB35jo1Yty/C2UO3Atr8SMeJFZMPTet8
40tD89dAZZU6eub0wD7GPM3EYSXjHl3ek9f5xzH7DQAA//8DAFBLAwQUAAYACAAAACEAvNkeN+QA
AAAQAQAADwAAAGRycy9kb3ducmV2LnhtbExPwU7DMAy9I/EPkZG4sXSpOtKu6YSGJu0ABwqIa9aY
tqJJSpNt3d9jTnCx/Ozn5/fKzWwHdsIp9N4pWC4SYOgab3rXKnh73d1JYCFqZ/TgHSq4YIBNdX1V
6sL4s3vBUx1bRiIuFFpBF+NYcB6aDq0OCz+io92nn6yOBKeWm0mfSdwOXCTJilvdO/rQ6RG3HTZf
9dEqeN7mtdyLy/SRp/tdLb+X/km+K3V7Mz+uqTysgUWc498F/GYg/1CRsYM/OhPYQDjNM6IqEFkm
gBFDyJQmB2ry+xXwquT/g1Q/AAAA//8DAFBLAQItABQABgAIAAAAIQC2gziS/gAAAOEBAAATAAAA
AAAAAAAAAAAAAAAAAABbQ29udGVudF9UeXBlc10ueG1sUEsBAi0AFAAGAAgAAAAhADj9If/WAAAA
lAEAAAsAAAAAAAAAAAAAAAAALwEAAF9yZWxzLy5yZWxzUEsBAi0AFAAGAAgAAAAhAFh0wPM/AgAA
gAQAAA4AAAAAAAAAAAAAAAAALgIAAGRycy9lMm9Eb2MueG1sUEsBAi0AFAAGAAgAAAAhALzZHjfk
AAAAEAEAAA8AAAAAAAAAAAAAAAAAmQQAAGRycy9kb3ducmV2LnhtbFBLBQYAAAAABAAEAPMAAACq
BQAAAAA=
" fillcolor="white [3201]" stroked="f" strokeweight=".5pt">
 <v:textbox>
  <![if !mso]>
  <table cellpadding=0 cellspacing=0 width="100%">
   <tr>
    <td><![endif]>
    <div>
    <p class=MsoNormal><span lang=EN-US style='color:red;mso-ansi-language:
    EN-US'>(2)<o:p></o:p></span></p>
    </div>
    <![if !mso]></td>
   </tr>
  </table>
  <![endif]></v:textbox>
</v:shape><![endif]--><![if !vml]><span style='mso-ignore:vglayout;position:
relative;z-index:251676672'><span style='position:absolute;left:60px;
top:-356px;width:32px;height:25px'><img width=32 height=25
src="/static/RAREFAN_Manual.fld/image039.png" alt="Text Box: (2)" v:shapes="Text_x0020_Box_x0020_19"></span></span><![endif]><span
lang=EN-US style='mso-ansi-language:EN-US;mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="Picture_x0020_4" o:spid="_x0000_i1029" type="#_x0000_t75" style='width:254pt;
 height:167pt;visibility:visible;mso-wrap-style:square' o:bordertopcolor="yellow pure"
 o:borderleftcolor="yellow pure" o:borderbottomcolor="yellow pure"
 o:borderrightcolor="yellow pure">
 <v:imagedata src="/static/RAREFAN_Manual.fld/image018.png" o:title=""/>
 <w:bordertop type="single" width="6"/>
 <w:borderleft type="single" width="6"/>
 <w:borderbottom type="single" width="6"/>
 <w:borderright type="single" width="6"/>
</v:shape><![endif]--><![if !vml]><img width=256 height=169
src="/static/RAREFAN_Manual.fld/image040.png" v:shapes="Picture_x0020_4"><![endif]></span><span
lang=EN-US style='mso-ansi-language:EN-US'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'>Once the
job is finished you can access your data either by browsing through the folder
structure online <span style='color:red'>(1)</span>; by downloading the data
and viewing it on your hard disk <span style='color:red'>(2)</span>; or by
directly plotting the data (see below for a description of the plots) <span
style='color:red'>(3).<o:p></o:p></span></span></p>

<p class=MsoNormal><span lang=EN-US style='color:red;mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-US style='mso-ansi-language:EN-US'>File output<o:p></o:p></span></h2>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>All
output data is located in a folder called </span><span style='font-family:Courier;
mso-bidi-font-family:Calibri;color:#333333'>out/</span><span style='font-family:
"Calibri",sans-serif;color:#333333'>.<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>The
output files include the following:<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>A
phylogenetic tree &quot;<span class=SpellE>tmptree.nwk</span>&quot; of all
genomes generated with <span class=SpellE>andi</span> (</span><span
style='color:black;mso-color-alt:windowtext'><a
href="http://github.com/evolbioinf/andi/"><span style='font-family:"Calibri",sans-serif;
mso-fareast-font-family:"Times New Roman";mso-fareast-theme-font:major-fareast;
color:black;mso-color-alt:windowtext'>http://github.com/evolbioinf/andi/</span></a></span><span
style='font-family:"Calibri",sans-serif;color:black;mso-color-alt:windowtext'>)
</span><span style='font-family:"Calibri",sans-serif;color:#333333'>and <span
class=SpellE>clustDist</span> </span><span style='font-family:"Calibri",sans-serif;
color:black;mso-color-alt:windowtext'>(</span><span style='color:black;
mso-color-alt:windowtext'><a
href="http://guanine.evolbio.mpg.de/problemsBook/node1.html"><span
style='font-family:"Calibri",sans-serif;color:black;mso-color-alt:windowtext'>http://guanine.evolbio.mpg.de/problemsBook/node1.html</span></a></span><span
style='font-family:"Calibri",sans-serif;color:#333333'>).<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>A
file containing the frequencies of all 21bp long sequences found in the designated
reference genome.<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>[reference]</span><span
style='font-family:Courier;mso-bidi-font-family:Calibri;color:#333333'>.<span
class=SpellE>overrep</span></span><span style='font-family:"Calibri",sans-serif;
color:#333333'> Contains all 21bp long sequences that occur more frequently
than n (default 55) times in the reference genome:&nbsp;<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span class=SpellE><span style='font-family:Courier;
mso-bidi-font-family:Calibri;color:#333333'>prox.stats</span></span><span
style='font-family:"Calibri",sans-serif;color:#333333'> Contains information on
the RAYTs and their cooccurrence with different REPIN populations. <o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>A
file containing the nucleotide sequences of all RAYT&nbsp;relatives identified
with BLAST+&nbsp;in all provided genomes:&nbsp;</span><span class=SpellE><span
style='font-family:Courier;mso-bidi-font-family:Calibri;color:#333333'>yafM_relatives.fna</span></span><span
style='font-family:Courier;mso-bidi-font-family:Calibri;color:#333333'>. </span><span
style='font-family:"Calibri",sans-serif;color:#333333'>Yet, only hits are put
out that are longer than 240 bp.</span><span style='font-family:Courier;
mso-bidi-font-family:Calibri;color:#333333'> </span><span style='font-family:
"Calibri",sans-serif;color:#333333'><o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span class=SpellE><span style='font-family:Courier;
mso-bidi-font-family:Calibri;color:#333333'>maxREPIN</span></span><span
style='font-family:Courier;mso-bidi-font-family:Calibri;color:#333333'>_</span><span
style='font-family:"Calibri",sans-serif;color:#333333'>[0-5] Contains the most
frequent REPIN identified for each sequence type in each&nbsp; strain.<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:Courier;mso-bidi-font-family:Calibri;
color:#333333'>presAbs_</span><span style='font-family:"Calibri",sans-serif;
color:#333333'>[0-5].txt Contains for each strain information on the number of
RAYTs, the number of REPINs, the master sequence, the number of master
sequences, the entire REP/REPIN population size, the number of REPIN clusters
that contain more than 10 sequences, all REPINs in the population as well as
all REPINs that differ to the master sequences in at most three nucleotides.<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span class=SpellE><span style='font-family:Courier;
mso-bidi-font-family:Calibri;color:#333333'>rayt</span></span><span
style='font-family:Courier;mso-bidi-font-family:Calibri;color:#333333'>_</span><span
style='font-family:"Calibri",sans-serif;color:#333333'>[strain name]</span><span
style='font-family:Courier;mso-bidi-font-family:Calibri;color:#333333'>.tab</span><span
style='font-family:"Calibri",sans-serif;color:#333333'> contains location
information for each identified RAYT relative for each strain. The files can be
viewed with <span class=SpellE>artemis</span> </span><!--[if supportFields]><span
style='font-family:"Calibri",sans-serif;color:#333333'><span style='mso-element:
field-begin'></span><span style='mso-spacerun:yes'> </span>ADDIN ZOTERO_ITEM
CSL_CITATION
{&quot;citationID&quot;:&quot;t7ked6Mr&quot;,&quot;properties&quot;:{&quot;formattedCitation&quot;:&quot;(5)&quot;,&quot;plainCitation&quot;:&quot;(5)&quot;,&quot;noteIndex&quot;:0},&quot;citationItems&quot;:[{&quot;id&quot;:391,&quot;uris&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/A8KUY5LR&quot;],&quot;uri&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/A8KUY5LR&quot;],&quot;itemData&quot;:{&quot;id&quot;:391,&quot;type&quot;:&quot;article-journal&quot;,&quot;abstract&quot;:&quot;SUMMARY:
Artemis is a DNA sequence visualization and annotation tool that allows the
results of any analysis or sets of analyses to be viewed in the context of the
sequence and its six-frame translation. Artemis is especially useful in
analysing the compact genomes of bacteria, archaea and lower eukaryotes, and
will cope with sequences of any size from small genes to whole genomes. It is
implemented in Java, and can be run on any suitable platform. Sequences and
annotation can be read and written directly in EMBL, GenBank and GFF format.
AVAILABITLTY: Artemis is available under the GNU General Public License from
http://www.sanger.ac.uk/Software/Artemis&quot;,&quot;container-title&quot;:&quot;Bioinformatics&quot;,&quot;issue&quot;:&quot;10&quot;,&quot;journalAbbreviation&quot;:&quot;Bioinformatics&quot;,&quot;page&quot;:&quot;944-945&quot;,&quot;title&quot;:&quot;Artemis:
sequence visualization and
annotation.&quot;,&quot;volume&quot;:&quot;16&quot;,&quot;author&quot;:[{&quot;family&quot;:&quot;Rutherford&quot;,&quot;given&quot;:&quot;K&quot;},{&quot;family&quot;:&quot;Parkhill&quot;,&quot;given&quot;:&quot;J&quot;},{&quot;family&quot;:&quot;Crook&quot;,&quot;given&quot;:&quot;J&quot;},{&quot;family&quot;:&quot;Horsnell&quot;,&quot;given&quot;:&quot;T&quot;},{&quot;family&quot;:&quot;Rice&quot;,&quot;given&quot;:&quot;P&quot;},{&quot;family&quot;:&quot;Rajandream&quot;,&quot;given&quot;:&quot;M
A&quot;},{&quot;family&quot;:&quot;Barrell&quot;,&quot;given&quot;:&quot;B&quot;}],&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2000&quot;,10,1]]}}}],&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}
<span style='mso-element:field-separator'></span></span><![endif]--><span
style='font-family:"Calibri",sans-serif;color:#333333'><span style='mso-no-proof:
yes'>(5)</span></span><!--[if supportFields]><span style='font-family:"Calibri",sans-serif;
color:#333333'><span style='mso-element:field-end'></span></span><![endif]--><span
style='font-family:"Calibri",sans-serif;color:#333333'>.<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:Courier;mso-bidi-font-family:Calibri;
color:#333333'>results.txt</span><span style='font-family:"Calibri",sans-serif;
color:#333333'> Contains for each strain the frequency of the six identified
21bp long seeds.<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>There
is one folder called </span><span class=SpellE><span style='font-family:Courier;
mso-bidi-font-family:Calibri;color:#333333'>groupSeedSequences</span></span><span
style='font-family:"Calibri",sans-serif;color:#333333'>, which includes the
data for identifying the most common 21 bp long sequences in&nbsp;</span><em><span
style='font-family:"Calibri",sans-serif;mso-fareast-font-family:"Times New Roman";
mso-fareast-theme-font:major-fareast;color:#333333;font-style:normal'>the
reference genome</span></em><span style='font-family:"Calibri",sans-serif;
color:#333333'>. All 21bp long sequences in the genome that occur more
frequently than 55 times are sorted into 6 sequence groups. These sequence
groups are stored in the files&nbsp;</span><span style='font-family:Courier;
mso-bidi-font-family:Calibri;color:#333333'>Group_</span><span
style='font-family:"Calibri",sans-serif;color:#333333'>[reference]</span><span
style='font-family:Courier;mso-bidi-font-family:Calibri;color:#333333'>_</span><span
style='font-family:"Calibri",sans-serif;color:#333333'>*</span><span
style='font-family:Courier;mso-bidi-font-family:Calibri;color:#333333'>.out</span><span
style='font-family:"Calibri",sans-serif;color:#333333'> and .</span><span
class=SpellE><span style='font-family:Courier;mso-bidi-font-family:Calibri;
color:#333333'>out.fas</span></span><span style='font-family:"Calibri",sans-serif;
color:#333333'>. There is also a [reference]</span><span style='font-family:
Courier;mso-bidi-font-family:Calibri;color:#333333'>_<span class=SpellE>words.tab</span></span><span
style='font-family:"Calibri",sans-serif;color:#333333'> file, which contains
the locations of all overrepresented 21bp long sequences in the reference
genome. This file can be viewed in <span class=SpellE>artemis</span>
(https://www.sanger.ac.uk/tool/artemis/) together with the reference genome
file. The most common sequence in each group is used as a seed sequence to
determine REPIN populations across all submitted sequence files.<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>&nbsp;<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>For
each genome there are six&nbsp;output folders (ending in </span><span
style='font-family:Courier;mso-bidi-font-family:Calibri;color:#333333'>_0</span><span
style='font-family:"Calibri",sans-serif;color:#333333'> to </span><span
style='font-family:Courier;mso-bidi-font-family:Calibri;color:#333333'>_5</span><span
style='font-family:"Calibri",sans-serif;color:#333333'>), for each sequence
group one.<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>Each
folder contains the following files:<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>*</span><span
style='font-family:Courier;mso-bidi-font-family:Calibri;color:#333333'>.dd</span><span
style='font-family:"Calibri",sans-serif;color:#333333'>: Degree distribution of
the REPIN network, where each REPIN is a node. A REPIN is connected to another
REPIN if they differ in exactly one position. The degree distribution is a
histogram of the number of connections of all the nodes.&nbsp;<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>*</span><span
style='font-family:Courier;mso-bidi-font-family:Calibri;color:#333333'>.hist</span><span
style='font-family:"Calibri",sans-serif;color:#333333'> For the largest
sequence cluster determined by mcl </span><!--[if supportFields]><span
style='font-family:"Calibri",sans-serif;color:#333333'><span style='mso-element:
field-begin'></span><span style='mso-spacerun:yes'> </span>ADDIN ZOTERO_ITEM
CSL_CITATION {&quot;citationID&quot;:&quot;mvmJNG1R&quot;,&quot;properties&quot;:{&quot;formattedCitation&quot;:&quot;(6)&quot;,&quot;plainCitation&quot;:&quot;(6)&quot;,&quot;noteIndex&quot;:0},&quot;citationItems&quot;:[{&quot;id&quot;:1201,&quot;uris&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/82ATPDV4&quot;],&quot;uri&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/82ATPDV4&quot;],&quot;itemData&quot;:{&quot;id&quot;:1201,&quot;type&quot;:&quot;article-journal&quot;,&quot;abstract&quot;:&quot;Detection
of protein families in large databases is one of the principal research
objectives in structural and functional genomics. Protein family classification
can significantly contribute to the delineation of functional diversity of
homologous proteins, the prediction of function based on domain architecture or
the presence of sequence motifs as well as comparative genomics, providing
valuable evolutionary insights. We present a novel approach called TRIBE-MCL
for rapid and accurate clustering of protein sequences into families. The
method relies on the Markov cluster (MCL) algorithm for the assignment of
proteins into families based on precomputed sequence similarity information.
This novel approach does not suffer from the problems that normally hinder
other protein sequence clustering algorithms, such as the presence of
multi-domain proteins, promiscuous domains and fragmented proteins. The method
has been rigorously tested and validated on a number of very large databases, including
SwissProt, InterPro, SCOP and the draft human genome. Our results indicate that
the method is ideally suited to the rapid and accurate detection of protein
families on a large scale. The method has been used to detect and categorise
protein families within the draft human genome and the resulting families have
been used to annotate a large proportion of human
proteins.&quot;,&quot;container-title&quot;:&quot;Nucleic Acids
Research&quot;,&quot;issue&quot;:&quot;7&quot;,&quot;journalAbbreviation&quot;:&quot;Nucleic
Acids Res.&quot;,&quot;note&quot;:&quot;publisher: Oxford University
Press&quot;,&quot;page&quot;:&quot;1575-1584&quot;,&quot;title&quot;:&quot;An
efficient algorithm for large-scale detection of protein
families.&quot;,&quot;volume&quot;:&quot;30&quot;,&quot;author&quot;:[{&quot;family&quot;:&quot;Enright&quot;,&quot;given&quot;:&quot;A
J&quot;},{&quot;family&quot;:&quot;Van
Dongen&quot;,&quot;given&quot;:&quot;S&quot;},{&quot;family&quot;:&quot;Ouzounis&quot;,&quot;given&quot;:&quot;C
A&quot;}],&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2002&quot;,4,1]]}}}],&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}
<span style='mso-element:field-separator'></span></span><![endif]--><span
style='font-family:"Calibri",sans-serif;color:#333333'><span style='mso-no-proof:
yes'>(6)</span></span><!--[if supportFields]><span style='font-family:"Calibri",sans-serif;
color:#333333'><span style='mso-element:field-end'></span></span><![endif]--><span
style='font-family:"Calibri",sans-serif;color:#333333'> that consists of REPINs
(two REPs in inverted orientation) this file contains the number of REPINs in
each sequence class. Sequence class 0 is the master sequence. By definition the
most common REPIN in the sequence population. Sequence class 1 contains all
REPINs differing in exactly one position to the master sequence. Sequence class
2 contains REPINs differing in 2 positions etc.<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>*</span><span
style='font-family:Courier;mso-bidi-font-family:Calibri;color:#333333'>.mcl</span><span
style='font-family:"Calibri",sans-serif;color:#333333'> Contains the clustering
output by mcl. Each line contains the member of a cluster. Lines are sorted by
cluster size.<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>*</span><span
style='font-family:Courier;mso-bidi-font-family:Calibri;color:#333333'>.mw</span><span
style='font-family:"Calibri",sans-serif;color:#333333'> Contains the most
common 21bp long sequence and its frequency in the genome, which is the basis
for identifying all related REP sequences and from those the REPINs formed by
these REP sequences.<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>*</span><span
style='font-family:Courier;mso-bidi-font-family:Calibri;color:#333333'>.nodes</span><span
style='font-family:"Calibri",sans-serif;color:#333333'> The identity and
frequency of all REPINs and REP sequences for&nbsp;either all sequences or only
for the largest sequence cluster.<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>*</span><span
style='font-family:Courier;mso-bidi-font-family:Calibri;color:#333333'>.ss</span><span
style='font-family:"Calibri",sans-serif;color:#333333'> Contains REPINs and REP
sequences as well as their positions in <span class=SpellE>fasta</span> format.
Position information starts with the location in genome <span class=SpellE>fasta</span>
file (first sequence is 0...) followed by the start and end position of the
entire REPIN/REP sequence.&nbsp;&nbsp;<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>*</span><span
style='font-family:Courier;mso-bidi-font-family:Calibri;color:#333333'>.<span
class=SpellE>ss.REP</span></span><span style='font-family:"Calibri",sans-serif;
color:#333333'> REP sequence information in <span class=SpellE>fasta</span>
format.<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>*</span><span
style='font-family:Courier;mso-bidi-font-family:Calibri;color:#333333'>.tab</span><span
style='font-family:"Calibri",sans-serif;color:#333333'> Location in tab format.
Can be used to display locations of REPs and REPINs in the genome via <span
class=SpellE>artemis</span>.<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>*_[0-9]</span><span
style='font-family:Courier;mso-bidi-font-family:Calibri;color:#333333'>.ss</span><span
style='font-family:"Calibri",sans-serif;color:#333333'> Contains REPIN/REP
sequence information for each subcluster separately.<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>*_[0-9]</span><span
style='font-family:Courier;mso-bidi-font-family:Calibri;color:#333333'>.tab</span><span
style='font-family:"Calibri",sans-serif;color:#333333'> Contains the location
of REP/REPINs for each subcluster separately for viewing in <span class=SpellE>artemis</span>.<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>*</span><span
class=SpellE><span style='font-family:Courier;mso-bidi-font-family:Calibri;
color:#333333'>allSeed.nw</span></span><span style='font-family:"Calibri",sans-serif;
color:#333333'> Contains network connections between nodes of all sequences.
Can be used to view network in for example R or <span class=SpellE>cytoscape</span>
together with the nodes file.<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>*</span><span
class=SpellE><span style='font-family:Courier;mso-bidi-font-family:Calibri;
color:#333333'>largestCluster.nodes</span></span><span style='font-family:"Calibri",sans-serif;
color:#333333'> Information on nodes only from the largest REPIN cluster.<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>*</span><span
class=SpellE><span style='font-family:Courier;mso-bidi-font-family:Calibri;
color:#333333'>largestCluster.ss</span></span><span style='font-family:"Calibri",sans-serif;
color:#333333'> *</span><span style='font-family:Courier;mso-bidi-font-family:
Calibri;color:#333333'>.ss</span><span style='font-family:"Calibri",sans-serif;
color:#333333'> file for the largest REPIN cluster.<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>*</span><span
class=SpellE><span style='font-family:Courier;mso-bidi-font-family:Calibri;
color:#333333'>largestCluster.tab</span></span><span style='font-family:"Calibri",sans-serif;
color:#333333'> *</span><span style='font-family:Courier;mso-bidi-font-family:
Calibri;color:#333333'>.tab</span><span style='font-family:"Calibri",sans-serif;
color:#333333'> file for the largest REPIN cluster.<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>*</span><span
style='font-family:Courier;mso-bidi-font-family:Calibri;color:#333333'>_rayt_repin_prox.txt</span><span
style='font-family:"Calibri",sans-serif;color:#333333'> shows which REPIN/REP
cluster is in proximity to any of the RAYT genes identified in the genome
(within 200bp).<o:p></o:p></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:7.5pt;margin-left:0cm;
background:white'><span style='font-family:"Calibri",sans-serif;color:#333333'>And
a subfolder that contains the complete sequences (including the variable
region) for all identified REPs and REPINs.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<h2><span lang=EN-US style='mso-ansi-language:EN-US'>Data Plots<o:p></o:p></span></h2>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US;mso-no-proof:
yes'><!--[if gte vml 1]><v:shape id="Picture_x0020_5" o:spid="_x0000_i1028"
 type="#_x0000_t75" style='width:234pt;height:149pt;visibility:visible;
 mso-wrap-style:square'>
 <v:imagedata src="/static/RAREFAN_Manual.fld/image020.png" o:title=""/>
</v:shape><![endif]--><![if !vml]><img border=0 width=234 height=149
src="/static/RAREFAN_Manual.fld/image020.png" v:shapes="Picture_x0020_5"><![endif]></span><span
lang=EN-US style='mso-ansi-language:EN-US'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='font-family:"Calibri",sans-serif;
mso-ansi-language:EN-US'>Here you can select the REPIN/RAYT type that is being plotted.
This is, for example, the data that is stored in the file <span class=SpellE>presAbs</span>_[number].txt.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US;mso-no-proof:
yes'><!--[if gte vml 1]><v:shape id="Picture_x0020_7" o:spid="_x0000_i1027"
 type="#_x0000_t75" style='width:469pt;height:213pt;visibility:visible;
 mso-wrap-style:square' o:bordertopcolor="yellow pure" o:borderleftcolor="yellow pure"
 o:borderbottomcolor="yellow pure" o:borderrightcolor="yellow pure">
 <v:imagedata src="/static/RAREFAN_Manual.fld/image021.png" o:title=""/>
 <w:bordertop type="single" width="6"/>
 <w:borderleft type="single" width="6"/>
 <w:borderbottom type="single" width="6"/>
 <w:borderright type="single" width="6"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=471 height=215
src="/static/RAREFAN_Manual.fld/image041.png" v:shapes="Picture_x0020_7"><![endif]></span><span
lang=EN-US style='mso-ansi-language:EN-US'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:-2.6pt;mso-layout-grid-align:none;
text-autospace:none'><span lang=EN-US style='font-family:"Calibri",sans-serif;
mso-ansi-language:EN-US'>The tree on the left side of the figure shows the
phylogeny of the submitted genomes. The tree was built applying neighbor
joining </span><!--[if supportFields]><span lang=EN-US style='font-family:"Calibri",sans-serif;
mso-ansi-language:EN-US'><span style='mso-element:field-begin'></span><span
style='mso-spacerun:yes'> </span>ADDIN ZOTERO_ITEM CSL_CITATION {&quot;citationID&quot;:&quot;fYAHkcoj&quot;,&quot;properties&quot;:{&quot;formattedCitation&quot;:&quot;(7)&quot;,&quot;plainCitation&quot;:&quot;(7)&quot;,&quot;noteIndex&quot;:0},&quot;citationItems&quot;:[{&quot;id&quot;:1036,&quot;uris&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/2IW55LF8&quot;],&quot;uri&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/2IW55LF8&quot;],&quot;itemData&quot;:{&quot;id&quot;:1036,&quot;type&quot;:&quot;article-journal&quot;,&quot;abstract&quot;:&quot;A
new method called the neighbor-joining method is proposed for reconstructing
phylogenetic trees from evolutionary distance data. The principle of this
method is to find pairs of operational taxonomic units (OTUs [= neighbors])
that minimize the total branch length at each stage of clustering of OTUs
starting with a starlike tree. The branch lengths as well as the topology of a
parsimonious tree can quickly be obtained by using this method. Using computer
simulation, we studied the efficiency of this method in obtaining the correct
unrooted tree in comparison with that of five other tree-making methods: the
unweighted pair group method of analysis, Farris's method, Sattath and
Tversky's method, Li's method, and Tateno et al.'s modified Farris method. The
new, neighbor-joining method and Sattath and Tversky's method are shown to be
generally better than the other
methods.&quot;,&quot;container-title&quot;:&quot;Molecular biology and
evolution&quot;,&quot;issue&quot;:&quot;4&quot;,&quot;journalAbbreviation&quot;:&quot;Mol.
Biol.
Evol.&quot;,&quot;page&quot;:&quot;406-425&quot;,&quot;title&quot;:&quot;The
neighbor-joining method: a new method for reconstructing phylogenetic
trees.&quot;,&quot;volume&quot;:&quot;4&quot;,&quot;author&quot;:[{&quot;family&quot;:&quot;Saitou&quot;,&quot;given&quot;:&quot;N&quot;},{&quot;family&quot;:&quot;Nei&quot;,&quot;given&quot;:&quot;M&quot;}],&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;1987&quot;,7,1]]}}}],&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}
<span style='mso-element:field-separator'></span></span><![endif]--><span
lang=EN-US style='font-family:"Calibri",sans-serif;mso-ansi-language:EN-US'><span
style='mso-no-proof:yes'>(7)</span></span><!--[if supportFields]><span
lang=EN-US style='font-family:"Calibri",sans-serif;mso-ansi-language:EN-US'><span
style='mso-element:field-end'></span></span><![endif]--><span lang=EN-US
style='font-family:"Calibri",sans-serif;mso-ansi-language:EN-US'> to a distance
matrix generated with the program “<span class=SpellE>andi</span>” applied to
whole genomes </span><!--[if supportFields]><span lang=EN-US style='font-family:
"Calibri",sans-serif;mso-ansi-language:EN-US'><span style='mso-element:field-begin'></span><span
style='mso-spacerun:yes'> </span>ADDIN ZOTERO_ITEM CSL_CITATION
{&quot;citationID&quot;:&quot;EisRqYac&quot;,&quot;properties&quot;:{&quot;formattedCitation&quot;:&quot;(2)&quot;,&quot;plainCitation&quot;:&quot;(2)&quot;,&quot;noteIndex&quot;:0},&quot;citationItems&quot;:[{&quot;id&quot;:156,&quot;uris&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/GZ74BZ5Z&quot;],&quot;uri&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/GZ74BZ5Z&quot;],&quot;itemData&quot;:{&quot;id&quot;:156,&quot;type&quot;:&quot;article-journal&quot;,&quot;abstract&quot;:&quot;MOTIVATION:A
standard approach to classifying sets of genomes is to calculate their pairwise
distances. This is difficult for large samples. We have therefore developed an
algorithm for rapidly computing the evolutionary distances between closely
related genomes.\n\nRESULTS:Our distance measure is based on ungapped local
alignments that we anchor through pairs of maximal unique matches of a minimum
length. These exact matches can be looked up efficiently using enhanced suffix
arrays and our implementation requires approximately only 1 s and 45 MB
RAM/Mbase analysed. The pairing of matches distinguishes non-homologous from
homologous regions leading to accurate distance estimation. We show this by
analysing simulated data and genome samples ranging from 29 Escherichia
coli/Shigella genomes to 3085 genomes of Streptococcus
pneumoniae.\n\nAVAILABILITY AND IMPLEMENTATION:We have implemented the
computation of anchor distances in the multithreaded UNIX command-line program
andi for ANchor DIstances. C sources and documentation are posted at
http://github.com/evolbioinf/andi/\n\nCONTACT:haubold@evolbio.mpg.de\n\nSUPPLEMENTARY
INFORMATION:Supplementary data are available at Bioinformatics
online.&quot;,&quot;container-title&quot;:&quot;Bioinformatics&quot;,&quot;DOI&quot;:&quot;10.1093/bioinformatics/btu815&quot;,&quot;issue&quot;:&quot;8&quot;,&quot;journalAbbreviation&quot;:&quot;Bioinformatics&quot;,&quot;page&quot;:&quot;1169-1175&quot;,&quot;title&quot;:&quot;andi:
fast and accurate estimation of evolutionary distances between closely related
genomes.&quot;,&quot;volume&quot;:&quot;31&quot;,&quot;author&quot;:[{&quot;family&quot;:&quot;Haubold&quot;,&quot;given&quot;:&quot;Bernhard&quot;},{&quot;family&quot;:&quot;Klötzl&quot;,&quot;given&quot;:&quot;Fabian&quot;},{&quot;family&quot;:&quot;Pfaffelhuber&quot;,&quot;given&quot;:&quot;Peter&quot;}],&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2015&quot;,4,15]]}}}],&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}
<span style='mso-element:field-separator'></span></span><![endif]--><span
lang=EN-US style='font-family:"Calibri",sans-serif;mso-ansi-language:EN-US'><span
style='mso-no-proof:yes'>(2)</span></span><!--[if supportFields]><span
lang=EN-US style='font-family:"Calibri",sans-serif;mso-ansi-language:EN-US'><span
style='mso-element:field-end'></span></span><![endif]--><span lang=EN-US
style='font-family:"Calibri",sans-serif;mso-ansi-language:EN-US'>. The next
column shows the presence and absence of the associated RAYT transposases. A
RAYT transposase is considered associated when a REPIN of the type is found
within 200bp of the transposase. The REPIN population size in each of the
genomes is shown in the last column. <o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US;mso-no-proof:
yes'><!--[if gte vml 1]><v:shape id="Picture_x0020_6" o:spid="_x0000_i1026"
 type="#_x0000_t75" style='width:468pt;height:217pt;visibility:visible;
 mso-wrap-style:square' o:bordertopcolor="yellow pure" o:borderleftcolor="yellow pure"
 o:borderbottomcolor="yellow pure" o:borderrightcolor="yellow pure">
 <v:imagedata src="/static/RAREFAN_Manual.fld/image023.png" o:title=""/>
 <w:bordertop type="single" width="6"/>
 <w:borderleft type="single" width="6"/>
 <w:borderbottom type="single" width="6"/>
 <w:borderright type="single" width="6"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=470 height=219
src="/static/RAREFAN_Manual.fld/image042.png" v:shapes="Picture_x0020_6"><![endif]></span><span
lang=EN-US style='mso-ansi-language:EN-US'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'>The second
plot shows the relationship between the RAYT genes. The tree was generated from
a multiple sequence alignment of RAYT DNA sequences using the program MUSCLE </span><!--[if supportFields]><span
lang=EN-US style='mso-ansi-language:EN-US'><span style='mso-element:field-begin'></span><span
style='mso-spacerun:yes'> </span>ADDIN ZOTERO_ITEM CSL_CITATION
{&quot;citationID&quot;:&quot;xaop90Qd&quot;,&quot;properties&quot;:{&quot;formattedCitation&quot;:&quot;(8)&quot;,&quot;plainCitation&quot;:&quot;(8)&quot;,&quot;noteIndex&quot;:0},&quot;citationItems&quot;:[{&quot;id&quot;:733,&quot;uris&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/UJUIMHRZ&quot;],&quot;uri&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/UJUIMHRZ&quot;],&quot;itemData&quot;:{&quot;id&quot;:733,&quot;type&quot;:&quot;article-journal&quot;,&quot;abstract&quot;:&quot;We
describe MUSCLE, a new computer program for creating multiple alignments of
protein sequences. Elements of the algorithm include fast distance estimation
using kmer counting, progressive alignment using a new profile function we call
the log-expectation score, and refinement using tree-dependent restricted
partitioning. The speed and accuracy of MUSCLE are compared with T-Coffee,
MAFFT and CLUSTALW on four test sets of reference alignments: BAliBASE,
SABmark, SMART and a new benchmark, PREFAB. MUSCLE achieves the highest, or
joint highest, rank in accuracy on each of these sets. Without refinement,
MUSCLE achieves average accuracy statistically indistinguishable from T-Coffee
and MAFFT, and is the fastest of the tested methods for large numbers of
sequences, aligning 5000 sequences of average length 350 in 7 min on a current
desktop computer. The MUSCLE program, source code and PREFAB test data are
freely available at http://www.drive5.
com/muscle.&quot;,&quot;container-title&quot;:&quot;Nucleic Acids Research&quot;,&quot;DOI&quot;:&quot;10.1093/nar/gkh340&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;journalAbbreviation&quot;:&quot;Nucleic
Acids Res.&quot;,&quot;note&quot;:&quot;publisher: Oxford University
Press&quot;,&quot;page&quot;:&quot;1792-1797&quot;,&quot;title&quot;:&quot;MUSCLE:
multiple sequence alignment with high accuracy and high
throughput.&quot;,&quot;volume&quot;:&quot;32&quot;,&quot;author&quot;:[{&quot;family&quot;:&quot;Edgar&quot;,&quot;given&quot;:&quot;Robert
C&quot;}],&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2004&quot;,1,1]]}}}],&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}
<span style='mso-element:field-separator'></span></span><![endif]--><span
lang=EN-US style='mso-ansi-language:EN-US'><span style='mso-no-proof:yes'>(8)</span></span><!--[if supportFields]><span
lang=EN-US style='mso-ansi-language:EN-US'><span style='mso-element:field-end'></span></span><![endif]--><span
lang=EN-US style='mso-ansi-language:EN-US'>. The tree itself is built using PHYML
</span><!--[if supportFields]><span lang=EN-US style='mso-ansi-language:EN-US'><span
style='mso-element:field-begin'></span><span
style='mso-spacerun:yes'> </span>ADDIN ZOTERO_ITEM CSL_CITATION
{&quot;citationID&quot;:&quot;iNAxYbzg&quot;,&quot;properties&quot;:{&quot;formattedCitation&quot;:&quot;(9)&quot;,&quot;plainCitation&quot;:&quot;(9)&quot;,&quot;noteIndex&quot;:0},&quot;citationItems&quot;:[{&quot;id&quot;:1447,&quot;uris&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/6BV3RMQD&quot;],&quot;uri&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/6BV3RMQD&quot;],&quot;itemData&quot;:{&quot;id&quot;:1447,&quot;type&quot;:&quot;article-journal&quot;,&quot;abstract&quot;:&quot;PhyML
is a phylogeny software based on the maximum-likelihood principle. Early PhyML
versions used a fast algorithm performing nearest neighbor interchanges to
improve a reasonable starting tree topology. Since the original publication
(Guindon S., Gascuel O. 2003. A simple, fast and accurate algorithm to estimate
large phylogenies by maximum likelihood. Syst. Biol. 52:696-704), PhyML has
been widely used (&gt;2500 citations in ISI Web of Science) because of its
simplicity and a fair compromise between accuracy and speed. In the meantime,
research around PhyML has continued, and this article describes the new
algorithms and methods implemented in the program. First, we introduce a new
algorithm to search the tree space with user-defined intensity using subtree
pruning and regrafting topological moves. The parsimony criterion is used here
to filter out the least promising topology modifications with respect to the
likelihood function. The analysis of a large collection of real nucleotide and
amino acid data sets of various sizes demonstrates the good performance of this
method. Second, we describe a new test to assess the support of the data for
internal branches of a phylogeny. This approach extends the recently proposed
approximate likelihood-ratio test and relies on a nonparametric,
Shimodaira-Hasegawa-like procedure. A detailed analysis of real alignments
sheds light on the links between this new approach and the more classical
nonparametric bootstrap method. Overall, our tests show that the last version
(3.0) of PhyML is fast, accurate, stable, and ready to use. A Web server and
binary files are available from
http://www.atgc-montpellier.fr/phyml/.&quot;,&quot;container-title&quot;:&quot;Systematic
Biology&quot;,&quot;DOI&quot;:&quot;10.1093/sysbio/syq010&quot;,&quot;issue&quot;:&quot;3&quot;,&quot;journalAbbreviation&quot;:&quot;Systematic
Biol.&quot;,&quot;note&quot;:&quot;publisher: Oxford University Press for the
Society of Systematic
Biologists&quot;,&quot;page&quot;:&quot;307-321&quot;,&quot;title&quot;:&quot;New
algorithms and methods to estimate maximum-likelihood phylogenies: assessing
the performance of PhyML 3.0.&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;author&quot;:[{&quot;family&quot;:&quot;Guindon&quot;,&quot;given&quot;:&quot;Stéphane&quot;},{&quot;family&quot;:&quot;Dufayard&quot;,&quot;given&quot;:&quot;Jean-François&quot;},{&quot;family&quot;:&quot;Lefort&quot;,&quot;given&quot;:&quot;Vincent&quot;},{&quot;family&quot;:&quot;Anisimova&quot;,&quot;given&quot;:&quot;Maria&quot;},{&quot;family&quot;:&quot;Hordijk&quot;,&quot;given&quot;:&quot;Wim&quot;},{&quot;family&quot;:&quot;Gascuel&quot;,&quot;given&quot;:&quot;Olivier&quot;}],&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2010&quot;,5,1]]}}}],&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}
<span style='mso-element:field-separator'></span></span><![endif]--><span
lang=EN-US style='mso-ansi-language:EN-US'><span style='mso-no-proof:yes'>(9)</span></span><!--[if supportFields]><span
lang=EN-US style='mso-ansi-language:EN-US'><span style='mso-element:field-end'></span></span><![endif]--><span
lang=EN-US style='mso-ansi-language:EN-US'>. The colors of the tip labels
correspond to the associated REPIN types. Colors are usually monophyletic due
to a strong association between RAYT and REPIN type.<o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US;mso-no-proof:
yes'><!--[if gte vml 1]><v:shape id="Picture_x0020_8" o:spid="_x0000_i1025"
 type="#_x0000_t75" style='width:468pt;height:210pt;visibility:visible;
 mso-wrap-style:square'>
 <v:imagedata src="/static/RAREFAN_Manual.fld/image025.png" o:title=""/>
</v:shape><![endif]--><![if !vml]><img border=0 width=468 height=210
src="/static/RAREFAN_Manual.fld/image025.png" v:shapes="Picture_x0020_8"><![endif]></span><span
lang=EN-US style='mso-ansi-language:EN-US'><o:p></o:p></span></p>

<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'>The
proportion of master sequences (indicates sequence conservation) in a REPIN
population and the REPIN population size. According to Quasispecies theory or
mutation-selection balance </span><!--[if supportFields]><span lang=EN-US
style='mso-ansi-language:EN-US'><span style='mso-element:field-begin'></span><span
style='mso-spacerun:yes'> </span>ADDIN ZOTERO_ITEM CSL_CITATION
{&quot;citationID&quot;:&quot;A3rZmngA&quot;,&quot;properties&quot;:{&quot;formattedCitation&quot;:&quot;(10,
11)&quot;,&quot;plainCitation&quot;:&quot;(10, 11)&quot;,&quot;noteIndex&quot;:0},&quot;citationItems&quot;:[{&quot;id&quot;:464,&quot;uris&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/E7XCP2ZZ&quot;],&quot;uri&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/E7XCP2ZZ&quot;],&quot;itemData&quot;:{&quot;id&quot;:464,&quot;type&quot;:&quot;article-journal&quot;,&quot;abstract&quot;:&quot;Mobile
genetic elements can be found in almost all genomes. Possibly the most common
nonautonomous mobile genetic elements in bacteria are repetitive extragenic
palindromic doublets forming hairpins (REPINs) that can occur hundreds of times
within a genome. The sum of all REPINs in a genome can be viewed as an evolving
population because REPINs replicate and mutate. In contrast to most other biological
populations, we know the exact composition of the REPIN population and the
sequence of each member of the population. Here, we model the evolution of
REPINs as quasispecies. We fit our quasispecies model to 10 different REPIN
populations from 10 different bacterial strains and estimate effective
duplication rates. Our estimated duplication rates range from </span><span
lang=EN-US style='font-family:"Cambria Math",serif;mso-bidi-font-family:"Cambria Math";
mso-ansi-language:EN-US'>∼</span><span lang=EN-US style='mso-ansi-language:
EN-US'>5 × 10-9 to 15 × 10-9 duplications per bacterial generation per REPIN.
The small range and the low level of the REPIN duplication rates suggest a universal
trade-off between the survival of the REPIN population and the reduction of the
mutational load for the host genome. The REPIN populations we investigated also
possess features typical of other natural populations. One population shows
hallmarks of a population that is going extinct, another population seems to be
growing in size, and we also see an example of competition between two REPIN
populations.&quot;,&quot;container-title&quot;:&quot;Genetics&quot;,&quot;DOI&quot;:&quot;10.1534/genetics.117.201160&quot;,&quot;issue&quot;:&quot;4&quot;,&quot;journalAbbreviation&quot;:&quot;Genetics&quot;,&quot;note&quot;:&quot;publisher:
Genetics&quot;,&quot;page&quot;:&quot;2149-2157&quot;,&quot;title&quot;:&quot;Discovering
Complete Quasispecies in Bacterial
Genomes.&quot;,&quot;volume&quot;:&quot;206&quot;,&quot;author&quot;:[{&quot;family&quot;:&quot;Bertels&quot;,&quot;given&quot;:&quot;Frederic&quot;},{&quot;family&quot;:&quot;Gokhale&quot;,&quot;given&quot;:&quot;Chaitanya
S&quot;},{&quot;family&quot;:&quot;Traulsen&quot;,&quot;given&quot;:&quot;Arne&quot;}],&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2017&quot;,8,1]]}}},{&quot;id&quot;:1564,&quot;uris&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/LQCHAJ24&quot;],&quot;uri&quot;:[&quot;http://zotero.org/users/local/hWv0kPbP/items/LQCHAJ24&quot;],&quot;itemData&quot;:{&quot;id&quot;:1564,&quot;type&quot;:&quot;report&quot;,&quot;abstract&quot;:&quot;Relationships
among organisms, in which one lives inside of another, with benefits accruing
to both partners, are referred to as endosymbiotic. Such relationships are
common in the biological world, yet descriptions are confined to organismal
life. Here we argue that short sequence repeats known as REPINs – whose
replication is dependent on a non-jumping RAYT transposase – form a similar
facultative symbiotic relationship with the bacterial chromosome. Evidence
stems from distribution patterns across the eubacteria: persistence times of
many millions of years, exceedingly rare duplication rates, vertical
transmission, and population biology typical of living organisms, including
population size fluctuations that correlate with available genome space.
Additional analysis of patterns of REPIN evolution conform with theoretical
predictions of conflict (and resolution) arising from the effects of selection
operating simultaneously on REPINs and host cells. A search for similar kinds
of genomic symbionts suggests that the REPIN-RAYT system is not
unique.&quot;,&quot;language&quot;:&quot;en&quot;,&quot;note&quot;:&quot;Company:
Cold Spring Harbor Laboratory\nDOI: 10.1101/2021.07.10.451892\nDistributor:
Cold Spring Harbor Laboratory\nLabel: Cold Spring Harbor Laboratory\nsection:
New Results\ntype: article&quot;,&quot;page&quot;:&quot;2021.07.10.451892&quot;,&quot;source&quot;:&quot;bioRxiv&quot;,&quot;title&quot;:&quot;REPINs
are facultative genomic symbionts of bacterial
genomes&quot;,&quot;URL&quot;:&quot;https://www.biorxiv.org/content/10.1101/2021.07.10.451892v1&quot;,&quot;author&quot;:[{&quot;family&quot;:&quot;Bertels&quot;,&quot;given&quot;:&quot;Frederic&quot;},{&quot;family&quot;:&quot;Rainey&quot;,&quot;given&quot;:&quot;Paul
B.&quot;}],&quot;accessed&quot;:{&quot;date-parts&quot;:[[&quot;2021&quot;,9,1]]},&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2021&quot;,7,11]]}}}],&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}
<span style='mso-element:field-separator'></span></span><![endif]--><span
lang=EN-US style='mso-ansi-language:EN-US'><span style='mso-no-proof:yes'>(10,
11)</span></span><!--[if supportFields]><span lang=EN-US style='mso-ansi-language:
EN-US'><span style='mso-element:field-end'></span></span><![endif]--><span
lang=EN-US style='mso-ansi-language:EN-US'>, the higher proportion of master
sequences (the most common sequence in the population) correlates with higher
duplication rates of the sequence population. The closer populations are to the
lower left of the plot, the smaller and more decayed they are and the less
likely they are to be alive (i.e. actively replicating). Only populations that
are colored are associated with a RAYT transposase. <o:p></o:p></span></p>

<h2><span lang=EN-US style='mso-ansi-language:EN-US'>References<o:p></o:p></span></h2>

<p class=MsoBibliography><!--[if supportFields]><span lang=EN-US
style='mso-ansi-language:EN-US;mso-fareast-language:EN-US'><span
style='mso-element:field-begin'></span><span
style='mso-spacerun:yes'> </span>ADDIN ZOTERO_BIBL
{&quot;uncited&quot;:[],&quot;omitted&quot;:[],&quot;custom&quot;:[]}
CSL_BIBLIOGRAPHY <span style='mso-element:field-separator'></span></span><![endif]-->1.
<span style='mso-tab-count:1'>  </span>F. Bertels, J. <span class=SpellE>Gallie</span>,
P. B. Rainey, Identification and Characterization of Domesticated Bacterial
Transposases. <i>Genome Biol. <span class=SpellE>Evol</span>.</i> <b>9</b>,
2110–2121 (2017).</p>

<p class=MsoBibliography>2. <span style='mso-tab-count:1'>  </span>B. <span
class=SpellE>Haubold</span>, F. <span class=SpellE>Klötzl</span>, P. <span
class=SpellE>Pfaffelhuber</span>, <span class=SpellE>andi</span>: fast and
accurate estimation of evolutionary distances between closely related genomes. <i>Bioinformatics</i>
<b>31</b>, 1169–1175 (2015).</p>

<p class=MsoBibliography>3. <span style='mso-tab-count:1'>  </span>F. Bertels,
P. B. Rainey, Within-Genome Evolution of REPINs: a New Family of Miniature
Mobile DNA in Bacteria. <span class=SpellE><i>PLoS</i></span><i> Genet.</i> <b>7</b>,
e1002132 (2011).</p>

<p class=MsoBibliography>4. <span style='mso-tab-count:1'>  </span>C. Camacho, <i>et
al.</i>, BLAST+: architecture and applications. <i>BMC Bioinformatics</i> <b>10</b>,
421–9 (2009).</p>

<p class=MsoBibliography>5. <span style='mso-tab-count:1'>  </span>K.
Rutherford, <i>et al.</i>, Artemis: sequence visualization and annotation. <i>Bioinformatics</i>
<b>16</b>, 944–945 (2000).</p>

<p class=MsoBibliography>6. <span style='mso-tab-count:1'>  </span>A. J.
Enright, S. Van <span class=SpellE>Dongen</span>, C. A. <span class=SpellE>Ouzounis</span>,
An efficient algorithm for large-scale detection of protein families. <i>Nucleic
Acids Res.</i> <b>30</b>, 1575–1584 (2002).</p>

<p class=MsoBibliography>7. <span style='mso-tab-count:1'>  </span>N. Saitou,
M. <span class=SpellE>Nei</span>, The <span class=SpellE>neighbor</span>-joining
method: a new method for reconstructing phylogenetic trees. <i>Mol. Biol. <span
class=SpellE>Evol</span>.</i> <b>4</b>, 406–425 (1987).</p>

<p class=MsoBibliography>8. <span style='mso-tab-count:1'>  </span>R. C. Edgar,
MUSCLE: multiple sequence alignment with high accuracy and high throughput. <i>Nucleic
Acids Res.</i> <b>32</b>, 1792–1797 (2004).</p>

<p class=MsoBibliography>9. <span style='mso-tab-count:1'>  </span>S. Guindon, <i>et
al.</i>, New algorithms and methods to estimate maximum-likelihood phylogenies:
assessing the performance of <span class=SpellE>PhyML</span> 3.0. <i>Syst.
Biol.</i> <b>59</b>, 307–321 (2010).</p>

<p class=MsoBibliography>10. <span style='mso-tab-count:1'> </span>F. Bertels,
C. S. Gokhale, A. <span class=SpellE>Traulsen</span>, Discovering Complete
Quasispecies in Bacterial Genomes. <i>Genetics</i> <b>206</b>, 2149–2157
(2017).</p>

<p class=MsoBibliography>11. <span style='mso-tab-count:1'> </span>F. Bertels,
P. B. Rainey, “REPINs are facultative genomic symbionts of bacterial genomes”
(2021).</p>

<p class=MsoNormal><!--[if supportFields]><span lang=EN-US style='mso-ansi-language:
EN-US;mso-fareast-language:EN-US'><span style='mso-element:field-end'></span></span><![endif]--><span
lang=EN-US style='mso-ansi-language:EN-US;mso-fareast-language:EN-US'><o:p>&nbsp;</o:p></span></p>

</div>

</body>
{% endblock %}

</html>
