<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<html>
<head>
	<meta http-equiv="content-type" content="text/html; charset=utf-8"/>
	<title></title>
	<meta name="generator" content="LibreOffice 7.0.4.2 (Linux)"/>
	<meta name="author" content="Frederic Bertels"/>
	<meta name="created" content="2021-08-31T08:42:00"/>
	<meta name="changedby" content="Frederic Bertels"/>
	<meta name="changed" content="2021-09-01T14:21:00"/>
	<meta name="AppVersion" content="16.0000"/>
	<meta name="ZOTERO_PREF_1" content="&lt;data data-version=&quot;3&quot; zotero-version=&quot;5.0.96.3&quot;&gt;&lt;session id=&quot;AMTsvI4E&quot;/&gt;&lt;style id=&quot;http://www.zotero.org/styles/pnas&quot; hasBibliography=&quot;1&quot; bibliographyStyleHasBeenSet=&quot;1&quot;/&gt;&lt;prefs&gt;&lt;pref name=&quot;fieldType&quot; value=&quot;Field&quot;/&gt;&lt;pref name=&quot;automaticJournalAbbreviati"/>
	<meta name="ZOTERO_PREF_2" content="ons&quot; value=&quot;true&quot;/&gt;&lt;/prefs&gt;&lt;/data&gt;"/>
	<style type="text/css">
		@page { size: 215.9mm 279.4mm; margin: 25.4mm }
		p { margin-bottom: 2.47mm; direction: ltr; line-height: 115%; text-align: left; orphans: 2; widows: 2; background: transparent }
		p.western { font-family: "Times New Roman", serif }
		p.cjk { font-family: "Times New Roman"; so-language: en-GB }
		p.ctl { font-family: "Times New Roman" }
		h1 { margin-bottom: 0mm; direction: ltr; color: #2f5496; line-height: 150%; text-align: left; page-break-inside: avoid; orphans: 2; widows: 2; background: transparent; page-break-after: avoid }
		h1.western { font-family: "Calibri Light", serif; font-size: 20pt }
		h1.cjk { font-family: ; font-size: 20pt; so-language: en-GB }
		h1.ctl { font-family: ; font-size: 16pt }
		h2 { margin-top: 0.71mm; margin-bottom: 0mm; direction: ltr; color: #2f5496; line-height: 150%; text-align: left; page-break-inside: avoid; orphans: 2; widows: 2; background: transparent; page-break-after: avoid }
		h2.western { font-family: "Calibri Light", serif; font-size: 14pt }
		h2.cjk { font-family: ; font-size: 14pt; so-language: en-US }
		h2.ctl { font-family: ; font-size: 13pt }
		em { font-style: italic }
		a:link { color: #0000ff; text-decoration: underline }
	</style>
</head>
<body lang="en-GB" link="#0000ff" vlink="#800000" dir="ltr"><h1 class="western">
<span lang="en-US">RAREFAN (RAyt/REpin Finder and ANalyzer) Manual</span></h1>
<p lang="en-US" class="western" style="margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<h2 class="western"><span lang="en-US">Using the online tool</span></h2>
<p class="western" style="margin-bottom: 0mm; line-height: 150%"><img src="/static/img/RAREFAN_Manual_html_a015251c61659e88.png" name="Picture 1" align="bottom" width="416" height="303" border="0"/>
</p>
<p class="western" style="margin-bottom: 0mm; line-height: 150%"><span lang="en-US">Here
you can upload the genomes that are to be analyzed for their REPIN
content. The genomes should be in fasta format and only have the
specified file extensions. </span>
</p>
<p lang="en-US" class="western" style="margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<p class="western" style="margin-bottom: 0mm; line-height: 150%"><span lang="en-US">You
can also upload RAYT query sequences. These should be fasta files
containing aminoacid sequences and should end in “.fa” or “.faa”.
If these are not provided then you can select a RAYT gene from </span><span lang="en-US"><i>P.
fluorescens </i></span><span lang="en-US">SBW25 (Group 3 RAYT (1))
or one from </span><span lang="en-US"><i>E. coli</i></span><span lang="en-US">
(Group 2 RAYT (1)) provided by the server on the next screen.</span></p>
<p lang="en-US" class="western" style="margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<p class="western" style="margin-bottom: 0mm; line-height: 150%"><span lang="en-US">Finally,
it is possible to upload a phylogenetic tree in Newick format for
your fasta sequences. For the webserver to be able to use the
provided tree the leaves of the tree have to match the sequence file
names. If the tree is not provided then a tree will be reconstructed
using andi (2). </span>
</p>
<p lang="en-US" class="western" style="margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<img src="/static/img/RAREFAN_Manual_html_aed9b8deedf22d17.gif" name="Shape8" alt="Shape8" align="left" vspace="1"/>

<img src="/static/img/RAREFAN_Manual_html_aed9b8deedf22d17.gif" name="Shape7" alt="Shape7" align="left" vspace="1"/>

<img src="/static/img/RAREFAN_Manual_html_aed9b8deedf22d17.gif" name="Shape6" alt="Shape6" align="left" vspace="1"/>

<img src="/static/img/RAREFAN_Manual_html_aed9b8deedf22d17.gif" name="Shape5" alt="Shape5" align="left" vspace="1"/>

<img src="/static/img/RAREFAN_Manual_html_aed9b8deedf22d17.gif" name="Shape4" alt="Shape4" align="left" vspace="1"/>

<img src="/static/img/RAREFAN_Manual_html_aed9b8deedf22d17.gif" name="Shape3" alt="Shape3" align="left" vspace="1"/>

<img src="/static/img/RAREFAN_Manual_html_aed9b8deedf22d17.gif" name="Shape2" alt="Shape2" align="left" vspace="1"/>

<img src="/static/img/RAREFAN_Manual_html_aed9b8deedf22d17.gif" name="Shape1" alt="Shape1" align="left" vspace="1"/>

<div id="Frame1" dir="ltr" style="float: left; width: 25.42mm; height: 8.68mm; border: none; padding: 0mm; background: #ffffff"><p style="margin-bottom: 0mm; line-height: 150%">
	<font color="#ff0000"><span lang="en-US">(8)</span></font></p>
</div>
<div id="Frame2" dir="ltr" style="float: left; width: 25.42mm; height: 8.68mm; border: none; padding: 0mm; background: #ffffff"><p style="margin-bottom: 0mm; line-height: 150%">
	<font color="#ff0000"><span lang="en-US">(7)</span></font></p>
</div>
<div id="Frame3" dir="ltr" style="float: left; width: 25.42mm; height: 8.68mm; border: none; padding: 0mm; background: #ffffff"><p style="margin-bottom: 0mm; line-height: 150%">
	<font color="#ff0000"><span lang="en-US">(6)</span></font></p>
</div>
<div id="Frame4" dir="ltr" style="float: left; width: 25.42mm; height: 6.67mm; border: none; padding: 0mm; background: #ffffff"><p style="margin-bottom: 0mm; line-height: 150%">
	<font color="#ff0000"><span lang="en-US">(5)</span></font></p>
</div>
<div id="Frame5" dir="ltr" style="float: left; width: 25.42mm; height: 8.68mm; border: none; padding: 0mm; background: #ffffff"><p style="margin-bottom: 0mm; line-height: 150%">
	<font color="#ff0000"><span lang="en-US">(4)</span></font></p>
</div>
<div id="Frame6" dir="ltr" style="float: left; width: 25.42mm; height: 8.68mm; border: none; padding: 0mm; background: #ffffff"><p style="margin-bottom: 0mm; line-height: 150%">
	<font color="#ff0000"><span lang="en-US">(3)</span></font></p>
</div>
<div id="Frame7" dir="ltr" style="float: left; width: 25.42mm; height: 8.68mm; border: none; padding: 0mm; background: #ffffff"><p style="margin-bottom: 0mm; line-height: 150%">
	<font color="#ff0000"><span lang="en-US">(2)</span></font></p>
</div>
<div id="Frame8" dir="ltr" style="float: left; width: 25.42mm; height: 8.68mm; border: none; padding: 0mm; background: #ffffff"><p style="margin-bottom: 0mm; line-height: 150%">
	<font color="#ff0000"><span lang="en-US">(1)</span></font></p>
</div>
<ol>
<!--	<li><p style="margin-bottom: 0mm; line-height: 150%"><span class="sd-abs-pos" style="position: absolute; top: 0mm; left: 0mm; width: 351px"><img src="/static/img/RAREFAN_Manual_html_e745a05dfbd4b57d.png" name="Picture 2" width="351" height="440" border="0"/>-->
</span><span lang="en-US">Please
	select a reference sequence from the sequence files you submitted.
	The reference sequence will be used to identify a maximum of six
	seed sequences. The seed sequences have to occur at least n times
	(specified in </span><font color="#ff0000"><span lang="en-US">(3)</span></font><span lang="en-US">)
	and are of length N (specified in </span><font color="#ff0000"><span lang="en-US">(5)</span></font><span lang="en-US">)
	and are the basis for determining sequence groups. The way sequence
	groups are created is described in detail here (3), in the
	section “Grouping of highly abundant oligonucleotides in SBW25”.
	If sequence groups are part of REPINs, then each sequence group can
	be used to define a REPIN type.</span></p>
</ol>
<p lang="en-US" class="western" style="margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<ol start="2">
	<li><p style="margin-bottom: 0mm; line-height: 150%"><span lang="en-US">There
	are two very divergent RAYT groups (</span><span lang="en-US"><i>E.
	coli </i></span><span lang="en-US"> group and </span><span lang="en-US"><i>P.
	fluorescens </i></span><span lang="en-US">group). If you do not know
	what kind of RAYT is present in your genome then you may have to run
	RAREFAN twice, once with the SBW25 and once with the </span><span lang="en-US"><i>E.
	coli </i></span><span lang="en-US">RAYT. If you have supplied a RAYT
	protein yourself you should be able to select it here. These
	sequences are then used as query proteins in a TBLASTN (4) search
	of the provided genomes. All sequences that are identified below the
	e-Value threshold set in </span><font color="#ff0000"><span lang="en-US">(6)</span></font><font color="#000000"><span lang="en-US">.</span></font></p>
</ol>
<p lang="en-US" style="margin-left: 12.7mm; margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<ol start="3">
	<li><p style="margin-bottom: 0mm; line-height: 150%"><span lang="en-US">If
	you have provided a tree file in the previous step then you can
	select it here.</span></p>
</ol>
<p lang="en-US" style="margin-left: 12.7mm; margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<ol start="4">
	<li><p style="margin-bottom: 0mm; line-height: 150%"><span lang="en-US">This
	parameter is required for identifying REPINs. Only sequences that
	occur more frequently than this value will be considered when
	identifying REPINs. </span>
	</p>
</ol>
<p lang="en-US" style="margin-left: 12.7mm; margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<ol start="5">
	<li><p style="margin-bottom: 0mm; line-height: 150%"><span lang="en-US">The
	seed sequence length for identifying REPINs. </span>
	</p>
</ol>
<p lang="en-US" style="margin-left: 12.7mm; margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<ol start="6">
	<li><p style="margin-bottom: 0mm; line-height: 150%"><span lang="en-US">This
	e-Value determines, which genes are identified as RAYTs in the
	genome.</span></p>
</ol>
<p lang="en-US" style="margin-left: 12.7mm; margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<ol start="7">
	<li><p style="margin-bottom: 0mm; line-height: 150%"><span lang="en-US">If
	this box is ticked then REPINs are identified. REPINs are defined as
	sequences that consist of two seed sequences in inverted orientation
	that are found at a distance of less than 130 bp. If the box is
	unticked then the seed sequences (REPs) identified as described in
	</span><font color="#ff0000"><span lang="en-US">(1)</span></font><span lang="en-US">
	are used for all further analyses. This can be useful if REPINs are
	asymmetric (i.e. there is for example a deletion/insertion in either
	the 5’ or 3’ REP sequence). This is for example the case for </span><span lang="en-US"><i>E.
	coli </i></span><span lang="en-US">REPINs.</span></p>
</ol>
<p lang="en-US" style="margin-left: 12.7mm; margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<ol start="8">
	<li><p style="margin-bottom: 0mm; line-height: 150%"><span lang="en-US">If
	you provide an email address here you will be notified once the job
	is finished. This is particular useful when applying RAREFAN to
	large datasets.</span></p>
</ol>
<p lang="en-US" class="western" style="margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<p class="western" style="margin-bottom: 0mm; line-height: 150%"><img src="/static/img/RAREFAN_Manual_html_9c500d6c4a440662.png" name="Picture 3" align="bottom" width="378" height="112" border="0"/>
</p>
<p lang="en-US" class="western" style="margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<p class="western" style="margin-bottom: 0mm; line-height: 150%"><span lang="en-US">Once
the job is submit, it is assigned a unique identifier. If you have
not supplied an email address you will either need to keep the site
open or remember the unique ID to access the website later. If you
have closed the website but know the unique ID you can access your
data by calling this address
http://rarefan.evolbio.mpg.de/results?run_id=</span></p>
<p lang="en-US" class="western" style="margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<p class="western" style="margin-bottom: 0mm; line-height: 150%"><img src="/static/img/RAREFAN_Manual_html_bbc6c766667a3e36.gif" name="Shape9" alt="Shape9" align="left"/>
<img src="/static/img/RAREFAN_Manual_html_bbc6c766667a3e36.gif" name="Shape11" alt="Shape11" align="left"/>
<img src="/static/img/RAREFAN_Manual_html_bbc6c766667a3e36.gif" name="Shape10" alt="Shape10" align="left"/>
<span id="Frame9" dir="ltr" style="float: left; width: 25.42mm; height: 7.5mm; border: none; padding: 0mm; background: #ffffff"><p style="margin-bottom: 0mm; line-height: 150%">
	<font color="#ff0000"><span lang="en-US">(1)</span></font></p>
</span>
<span id="Frame10" dir="ltr" style="float: left; width: 25.42mm; height: 7.5mm; border: none; padding: 0mm; background: #ffffff"><p style="margin-bottom: 0mm; line-height: 150%">
	<font color="#ff0000"><span lang="en-US">(3)</span></font></p>
</span>
<span id="Frame11" dir="ltr" style="float: left; width: 25.42mm; height: 7.5mm; border: none; padding: 0mm; background: #ffffff"><p style="margin-bottom: 0mm; line-height: 150%">
	<font color="#ff0000"><span lang="en-US">(2)</span></font></p>
</span><img src="/static/img/RAREFAN_Manual_html_5048dff53900e3fd.png" name="Picture 4" align="bottom" width="339" height="222" border="0"/>
</p>
<p lang="en-US" class="western" style="margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<p class="western" style="margin-bottom: 0mm; line-height: 150%"><span lang="en-US">Once
the job is finished you can access your data either by browsing
through the folder structure online </span><font color="#ff0000"><span lang="en-US">(1)</span></font><span lang="en-US">;
by downloading the data and viewing it on your hard disk </span><font color="#ff0000"><span lang="en-US">(2)</span></font><span lang="en-US">;
or by directly plotting the data (see below for a description of the
plots) </span><font color="#ff0000"><span lang="en-US">(3).</span></font></p>
<p lang="en-US" class="western" style="margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<h2 class="western"><span lang="en-US">File output</span></h2>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">All output data is
located in a folder called </font></font><font color="#333333"><font face="Courier, serif">out/</font></font><font color="#333333"><font face="Calibri, serif">.</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">The output files
include the following:</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">A phylogenetic tree
&quot;tmptree.nwk&quot; of all genomes generated with andi
(</font></font><font color="#0000ff"><u><a href="http://github.com/evolbioinf/andi/"><font color="#000000"><font face="Calibri, serif">http://github.com/evolbioinf/andi/</font></font></a></u></font><font face="Calibri, serif">)
</font><font color="#333333"><font face="Calibri, serif">and
clustDist </font></font><font face="Calibri, serif">(</font><font color="#0000ff"><u><a href="http://guanine.evolbio.mpg.de/problemsBook/node1.html"><font color="#000000"><font face="Calibri, serif">http://guanine.evolbio.mpg.de/problemsBook/node1.html</font></font></a></u></font><font color="#333333"><font face="Calibri, serif">).</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">A file containing
the frequencies of all 21bp long sequences found in the designated
reference genome.</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">[reference]</font></font><font color="#333333"><font face="Courier, serif">.overrep</font></font><font color="#333333"><font face="Calibri, serif">
Contains all 21bp long sequences that occur more frequently than n
(default 55) times in the reference genome:&nbsp;</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Courier, serif">prox.stats</font></font><font color="#333333"><font face="Calibri, serif">
Contains information on the RAYTs and their cooccurrence with
different REPIN populations. </font></font>
</p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">A file containing
the nucleotide sequences of all RAYT&nbsp;relatives identified with
BLAST+&nbsp;in all provided genomes:&nbsp;</font></font><font color="#333333"><font face="Courier, serif">yafM_relatives.fna</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Courier, serif">maxREPIN_</font></font><font color="#333333"><font face="Calibri, serif">[0-5]
Contains the most frequent REPIN identified for each sequence type in
each&nbsp; strain.</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Courier, serif">presAbs_</font></font><font color="#333333"><font face="Calibri, serif">[0-5].txt
Contains for each strain information on the number of RAYTs, the
number of REPINs, the master sequence, the number of master
sequences, the entire REP/REPIN population size, the number of REPIN
clusters that contain more than 10 sequences, all REPINs in the
population as well as all REPINs that differ to the master sequences
in at most three nucleotides.</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Courier, serif">rayt_</font></font><font color="#333333"><font face="Calibri, serif">[strain
name]</font></font><font color="#333333"><font face="Courier, serif">.tab</font></font><font color="#333333"><font face="Calibri, serif">
contains location information for each identified RAYT relative for
each strain. The files can be viewed with artemis (5).</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Courier, serif">results.txt</font></font><font color="#333333"><font face="Calibri, serif">
Contains for each strain the frequency of the six identified 21bp
long seeds.</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">There is one folder
called </font></font><font color="#333333"><font face="Courier, serif">groupSeedSequences</font></font><font color="#333333"><font face="Calibri, serif">,
which includes the data for identifying the most common 21 bp long
sequences in&nbsp;</font></font><em><font color="#333333"><font face="Calibri, serif"><span style="font-style: normal">the
reference genome</span></font></font></em><font color="#333333"><font face="Calibri, serif">.
All 21bp long sequences in the genome that occur more frequently than
55 times are sorted into 6 sequence groups. These sequence groups are
stored in the files&nbsp;</font></font><font color="#333333"><font face="Courier, serif">Group_</font></font><font color="#333333"><font face="Calibri, serif">[reference]</font></font><font color="#333333"><font face="Courier, serif">_</font></font><font color="#333333"><font face="Calibri, serif">*</font></font><font color="#333333"><font face="Courier, serif">.out</font></font><font color="#333333"><font face="Calibri, serif">
and .</font></font><font color="#333333"><font face="Courier, serif">out.fas</font></font><font color="#333333"><font face="Calibri, serif">.
There is also a [reference]</font></font><font color="#333333"><font face="Courier, serif">_words.tab</font></font><font color="#333333"><font face="Calibri, serif">
file, which contains the locations of all overrepresented 21bp long
sequences in the reference genome. This file can be viewed in artemis
(https://www.sanger.ac.uk/tool/artemis/) together with the reference
genome file. The most common sequence in each group is used as a seed
sequence to determine REPIN populations across all submitted sequence
files.</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333">&nbsp;</font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">For each genome
there are six&nbsp;output folders (ending in </font></font><font color="#333333"><font face="Courier, serif">_0</font></font><font color="#333333"><font face="Calibri, serif">
to </font></font><font color="#333333"><font face="Courier, serif">_5</font></font><font color="#333333"><font face="Calibri, serif">),
for each sequence group one.</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">Each folder
contains the following files:</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">*</font></font><font color="#333333"><font face="Courier, serif">.dd</font></font><font color="#333333"><font face="Calibri, serif">:
Degree distribution of the REPIN network, where each REPIN is a node.
A REPIN is connected to another REPIN if they differ in exactly one
position. The degree distribution is a histogram of the number of
connections of all the nodes.&nbsp;</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">*</font></font><font color="#333333"><font face="Courier, serif">.hist</font></font><font color="#333333"><font face="Calibri, serif">
For the largest sequence cluster determined by mcl (6) that
consists of REPINs (two REPs in inverted orientation) this file
contains the number of REPINs in each sequence class. Sequence class
0 is the master sequence. By definition the most common REPIN in the
sequence population. Sequence class 1 contains all REPINs differing
in exactly one position to the master sequence. Sequence class 2
contains REPINs differing in 2 positions etc.</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">*</font></font><font color="#333333"><font face="Courier, serif">.mcl</font></font><font color="#333333"><font face="Calibri, serif">
Contains the clustering output by mcl. Each line contains the member
of a cluster. Lines are sorted by cluster size.</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">*</font></font><font color="#333333"><font face="Courier, serif">.mw</font></font><font color="#333333"><font face="Calibri, serif">
Contains the most common 21bp long sequence and its frequency in the
genome, which is the basis for identifying all related REP sequences
and from those the REPINs formed by these REP sequences.</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">*</font></font><font color="#333333"><font face="Courier, serif">.nodes</font></font><font color="#333333"><font face="Calibri, serif">
The identity and frequency of all REPINs and REP sequences for&nbsp;either
all sequences or only for the largest sequence cluster.</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">*</font></font><font color="#333333"><font face="Courier, serif">.ss</font></font><font color="#333333"><font face="Calibri, serif">
Contains REPINs and REP sequences as well as their positions in fasta
format. Position information starts with the location in genome fasta
file (first sequence is 0...) followed by the start and end position
of the entire REPIN/REP sequence.&nbsp;&nbsp;</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">*</font></font><font color="#333333"><font face="Courier, serif">.ss.REP</font></font><font color="#333333"><font face="Calibri, serif">
REP sequence information in fasta format.</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">*</font></font><font color="#333333"><font face="Courier, serif">.tab</font></font><font color="#333333"><font face="Calibri, serif">
Location in tab format. Can be used to display locations of REPs and
REPINs in the genome via artemis.</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">*_[0-9]</font></font><font color="#333333"><font face="Courier, serif">.ss</font></font><font color="#333333"><font face="Calibri, serif">
Contains REPIN/REP sequence information for each subcluster
separately.</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">*_[0-9]</font></font><font color="#333333"><font face="Courier, serif">.tab</font></font><font color="#333333"><font face="Calibri, serif">
Contains the location of REP/REPINs for each subcluster separately
for viewing in artemis.</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">*</font></font><font color="#333333"><font face="Courier, serif">allSeed.nw</font></font><font color="#333333"><font face="Calibri, serif">
Contains network connections between nodes of all sequences. Can be
used to view network in for example R or cytoscape together with the
nodes file.</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">*</font></font><font color="#333333"><font face="Courier, serif">largestCluster.nodes</font></font><font color="#333333"><font face="Calibri, serif">
Information on nodes only from the largest REPIN cluster.</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">*</font></font><font color="#333333"><font face="Courier, serif">largestCluster.ss</font></font><font color="#333333"><font face="Calibri, serif">
*</font></font><font color="#333333"><font face="Courier, serif">.ss</font></font><font color="#333333"><font face="Calibri, serif">
file for the largest REPIN cluster.</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">*</font></font><font color="#333333"><font face="Courier, serif">largestCluster.tab</font></font><font color="#333333"><font face="Calibri, serif">
*</font></font><font color="#333333"><font face="Courier, serif">.tab</font></font><font color="#333333"><font face="Calibri, serif">
file for the largest REPIN cluster.</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">*</font></font><font color="#333333"><font face="Courier, serif">_rayt_repin_prox.txt</font></font><font color="#333333"><font face="Calibri, serif">
shows which REPIN/REP cluster is in proximity to any of the RAYT
genes identified in the genome (within 200bp).</font></font></p>
<p style="margin-bottom: 2.65mm; line-height: 150%; background: #ffffff">
<font color="#333333"><font face="Calibri, serif">And a subfolder
that contains the complete sequences (including the variable region)
for all identified REPs and REPINs.</font></font></p>
<p lang="en-US" class="western" style="margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<h2 class="western"><span lang="en-US">Data Plots</span></h2>
<p class="western" style="margin-bottom: 0mm; line-height: 150%"><img src="/static/img/RAREFAN_Manual_html_b2615fb9891bfcc0.png" name="Picture 5" align="bottom" width="311" height="198" border="0"/>
</p>
<p class="western" style="margin-bottom: 0mm; line-height: 150%"><font face="Calibri, serif"><span lang="en-US">Here
you can select the REPIN/RAYT type that is being plotted. This is,
for example, the data that is stored in the file
presAbs_[number].txt.</span></font></p>
<p class="western" style="margin-bottom: 0mm; line-height: 150%"><img src="/static/img/RAREFAN_Manual_html_400c3d23d3e4cfd7.png" name="Picture 7" align="bottom" width="624" height="284" border="0"/>
</p>
<p class="western" style="margin-right: -0.92mm; margin-bottom: 0mm; line-height: 150%">
<font face="Calibri, serif"><span lang="en-US">The tree on the left
side of the figure shows the phylogeny of the submitted genomes. The
tree was built applying neighbor joining (7) to a distance matrix
generated with the program “andi” applied to whole genomes
(2). The next column shows the presence and absence of the
associated RAYT transposases. A RAYT transposase is considered
associated when a REPIN of the type is found within 200bp of the
transposase. The REPIN population size in each of the genomes is
shown in the last column. </span></font>
</p>
<p lang="en-US" class="western" style="margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<p lang="en-US" class="western" style="margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<p lang="en-US" class="western" style="margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<p class="western" style="margin-bottom: 0mm; line-height: 150%"><img src="/static/img/RAREFAN_Manual_html_d463b78c41f5ac5b.png" name="Picture 6" align="bottom" width="624" height="289" border="0"/>
</p>
<p class="western" style="margin-bottom: 0mm; line-height: 150%"><span lang="en-US">The
second plot shows the relationship between the RAYT genes. The tree
was generated from a multiple sequence alignment of  RAYT </span><span lang="en-US">proteins</span><span lang="en-US">
using the program </span><span lang="en-US">XXX</span><span lang="en-US">.
The tree itself is built using </span><span lang="en-US">XXX</span><span lang="en-US">.
The colors of the tip labels correspond to the associated REPIN
types. Colors are usually monophyletic due to a strong association
between RAYT and REPIN type.</span></p>
<p lang="en-US" class="western" style="margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<p lang="en-US" class="western" style="margin-bottom: 0mm; line-height: 150%">
<br/>

</p>
<p class="western" style="margin-bottom: 0mm; line-height: 150%"><img src="/static/img/RAREFAN_Manual_html_4420efb3e263b5ed.png" name="Picture 8" align="bottom" width="624" height="280" border="0"/>
</p>
<p class="western" style="margin-bottom: 0mm; line-height: 150%"><span lang="en-US">The
proportion of master sequences (indicates sequence conservation) in a
REPIN population and the REPIN population size. According to
Quasispecies theory or mutation-selection balance (8, 9), the
higher proportion of master sequences (the most common sequence in
the population) correlates with higher duplication rates of the
sequence population. The closer populations are to the lower left of
the plot, the smaller and more decayed they are and the less likely
they are to be alive (i.e. actively replicating). Only populations
that are colored are associated with a RAYT transposase. </span>
</p>
<h2 class="western"><span lang="en-US">References</span></h2>
<p style="margin-left: 6.77mm; text-indent: -6.77mm; margin-bottom: 4.23mm; line-height: 150%">
1. 	F. Bertels, J. Gallie, P. B. Rainey, Identification and
Characterization of Domesticated Bacterial Transposases. <i>Genome
Biol. Evol.</i> <b>9</b>, 2110–2121 (2017).</p>
<p style="margin-left: 6.77mm; text-indent: -6.77mm; margin-bottom: 4.23mm; line-height: 150%">
2. 	B. Haubold, F. Klötzl, P. Pfaffelhuber, andi: fast and accurate
estimation of evolutionary distances between closely related genomes.
<i>Bioinformatics</i> <b>31</b>, 1169–1175 (2015).</p>
<p style="margin-left: 6.77mm; text-indent: -6.77mm; margin-bottom: 4.23mm; line-height: 150%">
3. 	F. Bertels, P. B. Rainey, Within-Genome Evolution of REPINs: a
New Family of Miniature Mobile DNA in Bacteria. <i>PLoS Genet.</i> <b>7</b>,
e1002132 (2011).</p>
<p style="margin-left: 6.77mm; text-indent: -6.77mm; margin-bottom: 4.23mm; line-height: 150%">
4. 	C. Camacho, <i>et al.</i>, BLAST+: architecture and applications.
<i>BMC Bioinformatics</i> <b>10</b>, 421–9 (2009).</p>
<p style="margin-left: 6.77mm; text-indent: -6.77mm; margin-bottom: 4.23mm; line-height: 150%">
5. 	K. Rutherford, <i>et al.</i>, Artemis: sequence visualization and
annotation. <i>Bioinformatics</i> <b>16</b>, 944–945 (2000).</p>
<p style="margin-left: 6.77mm; text-indent: -6.77mm; margin-bottom: 4.23mm; line-height: 150%">
6. 	A. J. Enright, S. Van Dongen, C. A. Ouzounis, An efficient
algorithm for large-scale detection of protein families. <i>Nucleic
Acids Res.</i> <b>30</b>, 1575–1584 (2002).</p>
<p style="margin-left: 6.77mm; text-indent: -6.77mm; margin-bottom: 4.23mm; line-height: 150%">
7. 	N. Saitou, M. Nei, The neighbor-joining method: a new method for
reconstructing phylogenetic trees. <i>Mol. Biol. Evol.</i> <b>4</b>,
406–425 (1987).</p>
<p style="margin-left: 6.77mm; text-indent: -6.77mm; margin-bottom: 4.23mm; line-height: 150%">
8. 	F. Bertels, C. S. Gokhale, A. Traulsen, Discovering Complete
Quasispecies in Bacterial Genomes. <i>Genetics</i> <b>206</b>,
2149–2157 (2017).</p>
<p style="margin-left: 6.77mm; text-indent: -6.77mm; margin-bottom: 4.23mm; line-height: 150%">
9. 	F. Bertels, P. B. Rainey, “REPINs are facultative genomic
symbionts of bacterial genomes” (2021).</p>
<p class="western" style="margin-bottom: 0mm; line-height: 150%"></p>
</body>
</html>